declare -x AZ_BATCH_ACCOUNT_NAME="exp1hildebatch"
declare -x AZ_BATCH_ACCOUNT_URL="https://exp1hildebatch.southeastasia.batch.azure.com/"
declare -x AZ_BATCH_CERTIFICATES_DIR="/mnt/batch/tasks/workitems/random_forest/job-1/task-00026/certs"
declare -x AZ_BATCH_JOB_ID="random_forest"
declare -x AZ_BATCH_JOB_PREP_DIR="/mnt/batch/tasks/workitems/random_forest/job-1/jobpreparation"
declare -x AZ_BATCH_JOB_PREP_WORKING_DIR="/mnt/batch/tasks/workitems/random_forest/job-1/jobpreparation/wd"
declare -x AZ_BATCH_NODE_ID="tvm-2112069986_6-20180411t200903z"
declare -x AZ_BATCH_NODE_IS_DEDICATED="true"
declare -x AZ_BATCH_NODE_ROOT_DIR="/mnt/batch/tasks"
declare -x AZ_BATCH_NODE_SHARED_DIR="/mnt/batch/tasks/shared"
declare -x AZ_BATCH_NODE_STARTUP_DIR="/mnt/batch/tasks/startup"
declare -x AZ_BATCH_POOL_ID="experiment"
declare -x AZ_BATCH_TASK_DIR="/mnt/batch/tasks/workitems/random_forest/job-1/task-00026"
declare -x AZ_BATCH_TASK_ID="task-00026"
declare -x AZ_BATCH_TASK_USER="_azbatch"
declare -x AZ_BATCH_TASK_USER_IDENTITY="PoolAdmin"
declare -x AZ_BATCH_TASK_WORKING_DIR="/mnt/batch/tasks/workitems/random_forest/job-1/task-00026/wd"
declare -x HOME="/mnt/batch/tasks/workitems/random_forest/job-1/task-00026/wd"
declare -x OLDPWD
declare -x PATH="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/mnt/batch/tasks/shared:/mnt/batch/tasks/workitems/random_forest/job-1/task-00026/wd"
declare -x PWD="/mnt/batch/tasks/workitems/random_forest/job-1/task-00026/wd"
declare -x SHLVL="1"
declare -x USER="_azbatch"
[2018-04-11 22:17:44] Results will not be logged locally.
[2018-04-11 22:17:44] Retrieving tasks...
[2018-04-11 22:17:44] Tasks will be retrieved based on provided task ids.
[2018-04-11 22:17:44] The following tasks were retrieved: [9981].
[2018-04-11 22:17:52] Downloaded task 9981.
[2018-04-11 22:17:53] Prepared pipelines.
[2018-04-11 22:17:53] Started run 1 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:17:53] Started run 3 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:17:53] Started run 2 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:17:53] Started run 4 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:17:53] Started run 5 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:17:53] Started run 7 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:17:53] Started run 8 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:17:53] Started run 6 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:17:57] Error in run 1: Run already exists in server. Run id(s): {9002356}
[2018-04-11 22:17:57] Started run 9 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:17:58] Error in run 3: Run already exists in server. Run id(s): {9002358}
[2018-04-11 22:17:58] Error in run 2: Run already exists in server. Run id(s): {9002366}
[2018-04-11 22:17:58] Started run 10 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:17:58] Started run 11 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:17:58] Error in run 4: Run already exists in server. Run id(s): {9002354}
[2018-04-11 22:17:58] Started run 12 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:17:58] Error in run 7: Run already exists in server. Run id(s): {9002365}
[2018-04-11 22:17:58] Error in run 5: Run already exists in server. Run id(s): {8941416}
[2018-04-11 22:17:58] Error in run 8: Run already exists in server. Run id(s): {9002360}
[2018-04-11 22:17:58] Started run 13 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:17:58] Started run 14 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:17:58] Error in run 6: Run already exists in server. Run id(s): {9002352}
[2018-04-11 22:17:58] Started run 15 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:17:58] Started run 16 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:02] Error in run 9: Run already exists in server. Run id(s): {9002353}
[2018-04-11 22:18:02] Started run 17 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:02] Error in run 10: Run already exists in server. Run id(s): {9002373}
[2018-04-11 22:18:02] Started run 18 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:02] Error in run 11: Run already exists in server. Run id(s): {9002380}
[2018-04-11 22:18:02] Started run 19 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:02] Error in run 12: Run already exists in server. Run id(s): {9002376}
[2018-04-11 22:18:02] Started run 20 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:03] Error in run 13: Run already exists in server. Run id(s): {9002382}
[2018-04-11 22:18:03] Error in run 14: Run already exists in server. Run id(s): {9002379}
[2018-04-11 22:18:03] Started run 21 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:03] Started run 22 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:03] Error in run 15: Run already exists in server. Run id(s): {9002370}
[2018-04-11 22:18:03] Error in run 16: Run already exists in server. Run id(s): {9002377}
[2018-04-11 22:18:03] Started run 23 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:03] Started run 24 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:07] Error in run 19: Run already exists in server. Run id(s): {9002395}
[2018-04-11 22:18:07] Error in run 20: Run already exists in server. Run id(s): {9002396}
[2018-04-11 22:18:07] Error in run 17: Run already exists in server. Run id(s): {9002389}
[2018-04-11 22:18:07] Error in run 18: Run already exists in server. Run id(s): {9002385}
[2018-04-11 22:18:07] Started run 25 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:07] Started run 26 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:07] Started run 27 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:07] Started run 28 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:08] Error in run 22: Run already exists in server. Run id(s): {9002392}
[2018-04-11 22:18:08] Error in run 24: Run already exists in server. Run id(s): {9002410}
[2018-04-11 22:18:08] Error in run 21: Run already exists in server. Run id(s): {9017267, 9002407}
[2018-04-11 22:18:08] Error in run 23: Run already exists in server. Run id(s): {9002394}
[2018-04-11 22:18:08] Started run 29 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:08] Started run 30 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:08] Started run 31 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:08] Started run 32 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:12] Error in run 25: Run already exists in server. Run id(s): {9002401}
[2018-04-11 22:18:12] Error in run 26: Run already exists in server. Run id(s): {9002406}
[2018-04-11 22:18:12] Started run 33 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:12] Started run 34 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:12] Error in run 27: Run already exists in server. Run id(s): {9002399}
[2018-04-11 22:18:12] Started run 35 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:12] Error in run 28: Run already exists in server. Run id(s): {9002411}
[2018-04-11 22:18:12] Started run 36 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:12] Error in run 29: Run already exists in server. Run id(s): {9002405}
[2018-04-11 22:18:12] Error in run 30: Run already exists in server. Run id(s): {9002448}
[2018-04-11 22:18:12] Started run 37 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:12] Error in run 31: Run already exists in server. Run id(s): {9002430}
[2018-04-11 22:18:12] Started run 38 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:12] Started run 39 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:13] Error in run 32: Run already exists in server. Run id(s): {9017314, 9002442}
[2018-04-11 22:18:13] Started run 40 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:16] Error in run 33: Run already exists in server. Run id(s): {9002415}
[2018-04-11 22:18:16] Started run 41 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:16] Error in run 34: Run already exists in server. Run id(s): {9002426}
[2018-04-11 22:18:16] Started run 42 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:16] Error in run 35: Run already exists in server. Run id(s): {9002420}
[2018-04-11 22:18:16] Error in run 36: Run already exists in server. Run id(s): {9002450}
[2018-04-11 22:18:16] Started run 43 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:16] Started run 44 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:17] Error in run 37: Run already exists in server. Run id(s): {9002431}
[2018-04-11 22:18:17] Error in run 38: Run already exists in server. Run id(s): {9002438}
[2018-04-11 22:18:17] Started run 45 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:17] Error in run 39: Run already exists in server. Run id(s): {9002477}
[2018-04-11 22:18:17] Started run 46 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:17] Started run 47 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:17] Error in run 40: Run already exists in server. Run id(s): {9002445}
[2018-04-11 22:18:17] Started run 48 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:21] Error in run 41: Run already exists in server. Run id(s): {9002473}
[2018-04-11 22:18:21] Started run 49 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:21] Error in run 42: Run already exists in server. Run id(s): {9002478}
[2018-04-11 22:18:21] Started run 50 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:21] Error in run 43: Run already exists in server. Run id(s): {9002498}
[2018-04-11 22:18:21] Started run 51 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:21] Error in run 44: Run already exists in server. Run id(s): {9002493}
[2018-04-11 22:18:21] Started run 52 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:22] Error in run 45: Run already exists in server. Run id(s): {9002488}
[2018-04-11 22:18:22] Error in run 46: Run already exists in server. Run id(s): {9002483}
[2018-04-11 22:18:22] Started run 53 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:22] Error in run 47: Run already exists in server. Run id(s): {9002467}
[2018-04-11 22:18:22] Started run 54 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:22] Error in run 48: Run already exists in server. Run id(s): {9002491}
[2018-04-11 22:18:22] Started run 55 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:22] Started run 56 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:25] Error in run 49: Run already exists in server. Run id(s): {9002523}
[2018-04-11 22:18:25] Started run 57 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:25] Error in run 50: Run already exists in server. Run id(s): {9002509}
[2018-04-11 22:18:25] Error in run 51: Run already exists in server. Run id(s): {9002520}
[2018-04-11 22:18:25] Started run 58 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:26] Error in run 52: Run already exists in server. Run id(s): {9002528}
[2018-04-11 22:18:25] Started run 59 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:26] Started run 60 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:26] Error in run 53: Run already exists in server. Run id(s): {9002524}
[2018-04-11 22:18:26] Error in run 54: Run already exists in server. Run id(s): {9002537}
[2018-04-11 22:18:26] Error in run 55: Run already exists in server. Run id(s): {9002536}
[2018-04-11 22:18:26] Started run 61 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:26] Error in run 56: Run already exists in server. Run id(s): {9002531}
[2018-04-11 22:18:26] Started run 62 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:26] Started run 64 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:26] Started run 63 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:30] Error in run 57: Run already exists in server. Run id(s): {9002560}
[2018-04-11 22:18:30] Started run 65 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:30] Error in run 59: Run already exists in server. Run id(s): {9002548}
[2018-04-11 22:18:30] Error in run 58: Run already exists in server. Run id(s): {9002587}
[2018-04-11 22:18:30] Error in run 60: Run already exists in server. Run id(s): {9002553}
[2018-04-11 22:18:30] Started run 66 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:30] Started run 68 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:30] Started run 67 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:31] Error in run 61: Run already exists in server. Run id(s): {9002611}
[2018-04-11 22:18:31] Error in run 62: Run already exists in server. Run id(s): {9002580}
[2018-04-11 22:18:31] Started run 69 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:31] Error in run 64: Run already exists in server. Run id(s): {9002577}
[2018-04-11 22:18:31] Started run 70 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:31] Started run 71 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:31] Error in run 63: Run already exists in server. Run id(s): {9002581}
[2018-04-11 22:18:31] Started run 72 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:34] Error in run 65: Run already exists in server. Run id(s): {9002591}
[2018-04-11 22:18:34] Started run 73 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:35] Error in run 66: Run already exists in server. Run id(s): {9002575}
[2018-04-11 22:18:35] Error in run 68: Run already exists in server. Run id(s): {9002616}
[2018-04-11 22:18:35] Started run 74 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:35] Error in run 67: Run already exists in server. Run id(s): {9002593}
[2018-04-11 22:18:35] Started run 75 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:35] Started run 76 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:36] Error in run 69: Run already exists in server. Run id(s): {9002618}
[2018-04-11 22:18:36] Error in run 70: Run already exists in server. Run id(s): {9002626}
[2018-04-11 22:18:36] Started run 77 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:36] Error in run 71: Run already exists in server. Run id(s): {9002614}
[2018-04-11 22:18:36] Started run 78 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:36] Error in run 72: Run already exists in server. Run id(s): {9002608}
[2018-04-11 22:18:36] Started run 80 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:36] Started run 79 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:39] Error in run 73: Run already exists in server. Run id(s): {9002636}
[2018-04-11 22:18:39] Started run 81 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:39] Error in run 74: Run already exists in server. Run id(s): {9002648}
[2018-04-11 22:18:39] Started run 82 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:40] Error in run 75: Run already exists in server. Run id(s): {9002651}
[2018-04-11 22:18:40] Error in run 76: Run already exists in server. Run id(s): {9002638}
[2018-04-11 22:18:40] Started run 83 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:40] Started run 84 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:40] Error in run 77: Run already exists in server. Run id(s): {9002677}
[2018-04-11 22:18:40] Started run 85 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:40] Error in run 78: Run already exists in server. Run id(s): {9002633}
[2018-04-11 22:18:40] Error in run 80: Run already exists in server. Run id(s): {9002674}
[2018-04-11 22:18:40] Started run 86 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:40] Error in run 79: Run already exists in server. Run id(s): {9002652}
[2018-04-11 22:18:40] Started run 87 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:40] Started run 88 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:43] Error in run 81: Run already exists in server. Run id(s): {9002657}
[2018-04-11 22:18:43] Started run 89 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:44] Error in run 82: Run already exists in server. Run id(s): {9002701}
[2018-04-11 22:18:44] Started run 90 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:44] Error in run 83: Run already exists in server. Run id(s): {9002661}
[2018-04-11 22:18:44] Error in run 84: Run already exists in server. Run id(s): {9002684}
[2018-04-11 22:18:44] Started run 91 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:44] Started run 92 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:45] Error in run 85: Run already exists in server. Run id(s): {9002672}
[2018-04-11 22:18:45] Started run 93 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:45] Error in run 86: Run already exists in server. Run id(s): {9002722}
[2018-04-11 22:18:45] Started run 94 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:45] Error in run 87: Run already exists in server. Run id(s): {9002706}
[2018-04-11 22:18:45] Error in run 88: Run already exists in server. Run id(s): {9002691}
[2018-04-11 22:18:45] Started run 95 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:45] Started run 96 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:48] Error in run 89: Run already exists in server. Run id(s): {9002713}
[2018-04-11 22:18:48] Started run 97 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:49] Error in run 90: Run already exists in server. Run id(s): {9002704}
[2018-04-11 22:18:49] Started run 98 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:49] Error in run 91: Run already exists in server. Run id(s): {9002698}
[2018-04-11 22:18:49] Started run 99 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:49] Error in run 92: Run already exists in server. Run id(s): {9002731}
[2018-04-11 22:18:49] Started run 100 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:49] Error in run 93: Run already exists in server. Run id(s): {9002714}
[2018-04-11 22:18:49] Started run 101 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:50] Error in run 94: Run already exists in server. Run id(s): {9002732}
[2018-04-11 22:18:50] Started run 102 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:50] Error in run 95: Run already exists in server. Run id(s): {9002726}
[2018-04-11 22:18:50] Error in run 96: Run already exists in server. Run id(s): {9002738}
[2018-04-11 22:18:50] Started run 103 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:50] Started run 104 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:53] Error in run 97: Run already exists in server. Run id(s): {9002730}
[2018-04-11 22:18:53] Started run 105 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:53] Error in run 98: Run already exists in server. Run id(s): {9002748}
[2018-04-11 22:18:53] Started run 106 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:53] Error in run 99: Run already exists in server. Run id(s): {9002777}
[2018-04-11 22:18:53] Started run 107 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:54] Error in run 100: Run already exists in server. Run id(s): {9002750}
[2018-04-11 22:18:54] Started run 108 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:54] Error in run 101: Run already exists in server. Run id(s): {9002768}
[2018-04-11 22:18:54] Started run 109 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:54] Error in run 102: Run already exists in server. Run id(s): {9002791}
[2018-04-11 22:18:54] Started run 110 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:54] Error in run 103: Run already exists in server. Run id(s): {9002754}
[2018-04-11 22:18:54] Started run 111 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:54] Error in run 104: Run already exists in server. Run id(s): {9002761}
[2018-04-11 22:18:54] Started run 112 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:57] Error in run 105: Run already exists in server. Run id(s): {9002771}
[2018-04-11 22:18:57] Started run 113 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:58] Error in run 106: Run already exists in server. Run id(s): {9002782}
[2018-04-11 22:18:58] Started run 114 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:58] Error in run 107: Run already exists in server. Run id(s): {9002804}
[2018-04-11 22:18:58] Started run 115 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:58] Error in run 108: Run already exists in server. Run id(s): {9002806}
[2018-04-11 22:18:58] Started run 116 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:58] Error in run 109: Run already exists in server. Run id(s): {9002811}
[2018-04-11 22:18:58] Started run 117 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:59] Error in run 110: Run already exists in server. Run id(s): {9002808}
[2018-04-11 22:18:59] Started run 118 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:59] Error in run 111: Run already exists in server. Run id(s): {9002825}
[2018-04-11 22:18:59] Started run 119 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:18:59] Error in run 112: Run already exists in server. Run id(s): {9002859}
[2018-04-11 22:18:59] Started run 120 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:02] Error in run 113: Run already exists in server. Run id(s): {9002840}
[2018-04-11 22:19:02] Started run 121 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:02] Error in run 114: Run already exists in server. Run id(s): {9002821}
[2018-04-11 22:19:02] Started run 122 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:02] Error in run 115: Run already exists in server. Run id(s): {9002830}
[2018-04-11 22:19:02] Started run 123 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:03] Error in run 116: Run already exists in server. Run id(s): {9002831}
[2018-04-11 22:19:03] Started run 124 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:03] Error in run 117: Run already exists in server. Run id(s): {9002902}
[2018-04-11 22:19:03] Started run 125 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:03] Error in run 118: Run already exists in server. Run id(s): {9002845}
[2018-04-11 22:19:03] Started run 126 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:03] Error in run 119: Run already exists in server. Run id(s): {9002879}
[2018-04-11 22:19:03] Started run 127 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:04] Error in run 120: Run already exists in server. Run id(s): {9002858}
[2018-04-11 22:19:04] Started run 128 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:06] Error in run 121: Run already exists in server. Run id(s): {9002869}
[2018-04-11 22:19:06] Started run 129 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:07] Error in run 122: Run already exists in server. Run id(s): {9002877}
[2018-04-11 22:19:07] Error in run 123: Run already exists in server. Run id(s): {9002868}
[2018-04-11 22:19:07] Started run 130 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:07] Started run 131 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:07] Error in run 124: Run already exists in server. Run id(s): {9002875}
[2018-04-11 22:19:07] Started run 132 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:08] Error in run 125: Run already exists in server. Run id(s): {9002912}
[2018-04-11 22:19:08] Started run 133 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:08] Error in run 126: Run already exists in server. Run id(s): {9002922}
[2018-04-11 22:19:08] Started run 134 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:08] Error in run 127: Run already exists in server. Run id(s): {9002906}
[2018-04-11 22:19:08] Started run 135 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:08] Error in run 128: Run already exists in server. Run id(s): {9002898}
[2018-04-11 22:19:08] Started run 136 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:11] Error in run 129: Run already exists in server. Run id(s): {9002890}
[2018-04-11 22:19:11] Started run 137 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:12] Error in run 130: Run already exists in server. Run id(s): {9002940}
[2018-04-11 22:19:12] Error in run 131: Run already exists in server. Run id(s): {9002910}
[2018-04-11 22:19:12] Started run 138 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:12] Started run 139 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:12] Error in run 132: Run already exists in server. Run id(s): {9002937}
[2018-04-11 22:19:12] Started run 140 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:12] Error in run 133: Run already exists in server. Run id(s): {9002925}
[2018-04-11 22:19:12] Started run 141 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:12] Error in run 134: Run already exists in server. Run id(s): {9002947}
[2018-04-11 22:19:12] Started run 142 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:13] Error in run 135: Run already exists in server. Run id(s): {9002983}
[2018-04-11 22:19:13] Started run 143 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:13] Error in run 136: Run already exists in server. Run id(s): {9002961}
[2018-04-11 22:19:13] Started run 144 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:16] Error in run 137: Run already exists in server. Run id(s): {9002950}
[2018-04-11 22:19:16] Started run 145 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:16] Error in run 138: Run already exists in server. Run id(s): {9002941}
[2018-04-11 22:19:16] Error in run 139: Run already exists in server. Run id(s): {9002959}
[2018-04-11 22:19:16] Error in run 140: Run already exists in server. Run id(s): {9002979}
[2018-04-11 22:19:16] Started run 146 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:16] Started run 147 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:17] Error in run 141: Run already exists in server. Run id(s): {9002970}
[2018-04-11 22:19:16] Started run 148 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:17] Started run 149 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:17] Error in run 142: Run already exists in server. Run id(s): {9002964}
[2018-04-11 22:19:17] Error in run 143: Run already exists in server. Run id(s): {9002977}
[2018-04-11 22:19:17] Started run 150 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:17] Started run 151 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:17] Error in run 144: Run already exists in server. Run id(s): {9002986}
[2018-04-11 22:19:17] Started run 152 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:20] Error in run 145: Run already exists in server. Run id(s): {9003011}
[2018-04-11 22:19:20] Started run 153 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:21] Error in run 146: Run already exists in server. Run id(s): {9003006}
[2018-04-11 22:19:21] Error in run 147: Run already exists in server. Run id(s): {9003003}
[2018-04-11 22:19:21] Started run 154 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:21] Error in run 148: Run already exists in server. Run id(s): {9002997}
[2018-04-11 22:19:21] Started run 155 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:21] Error in run 149: Run already exists in server. Run id(s): {9003017}
[2018-04-11 22:19:21] Started run 156 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:21] Started run 157 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:22] Error in run 150: Run already exists in server. Run id(s): {9002998}
[2018-04-11 22:19:22] Error in run 151: Run already exists in server. Run id(s): {9003033}
[2018-04-11 22:19:22] Started run 158 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:22] Started run 159 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:22] Error in run 152: Run already exists in server. Run id(s): {9003020}
[2018-04-11 22:19:22] Started run 160 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:25] Error in run 153: Run already exists in server. Run id(s): {9003041}
[2018-04-11 22:19:25] Started run 161 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:26] Error in run 155: Run already exists in server. Run id(s): {9003043}
[2018-04-11 22:19:26] Error in run 154: Run already exists in server. Run id(s): {9003027}
[2018-04-11 22:19:26] Error in run 156: Run already exists in server. Run id(s): {9003029}
[2018-04-11 22:19:26] Started run 162 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:26] Error in run 157: Run already exists in server. Run id(s): {9003030}
[2018-04-11 22:19:26] Started run 164 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:26] Started run 163 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:26] Started run 165 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:27] Error in run 159: Run already exists in server. Run id(s): {9003069}
[2018-04-11 22:19:27] Error in run 158: Run already exists in server. Run id(s): {9003065}
[2018-04-11 22:19:27] Started run 166 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:27] Error in run 160: Run already exists in server. Run id(s): {9003101}
[2018-04-11 22:19:27] Started run 167 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:27] Started run 168 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:29] Error in run 161: Run already exists in server. Run id(s): {9003058}
[2018-04-11 22:19:29] Started run 169 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:30] Error in run 162: Run already exists in server. Run id(s): {9003062}
[2018-04-11 22:19:30] Error in run 164: Run already exists in server. Run id(s): {9003067}
[2018-04-11 22:19:30] Started run 170 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:31] Error in run 163: Run already exists in server. Run id(s): {9003060}
[2018-04-11 22:19:30] Started run 171 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:31] Error in run 165: Run already exists in server. Run id(s): {9003073}
[2018-04-11 22:19:31] Started run 172 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:31] Started run 173 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:31] Error in run 166: Run already exists in server. Run id(s): {9003131}
[2018-04-11 22:19:31] Error in run 167: Run already exists in server. Run id(s): {9003088}
[2018-04-11 22:19:31] Started run 174 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:31] Error in run 168: Run already exists in server. Run id(s): {9003130}
[2018-04-11 22:19:31] Started run 175 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:31] Started run 176 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:34] Error in run 169: Run already exists in server. Run id(s): {9003086}
[2018-04-11 22:19:34] Started run 177 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:35] Error in run 170: Run already exists in server. Run id(s): {9003128}
[2018-04-11 22:19:35] Started run 178 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:35] Error in run 172: Run already exists in server. Run id(s): {9003114}
[2018-04-11 22:19:35] Error in run 171: Run already exists in server. Run id(s): {9003135}
[2018-04-11 22:19:35] Started run 179 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:35] Error in run 173: Run already exists in server. Run id(s): {9003105}
[2018-04-11 22:19:35] Started run 181 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:35] Started run 180 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:36] Error in run 175: Run already exists in server. Run id(s): {9003150}
[2018-04-11 22:19:36] Error in run 174: Run already exists in server. Run id(s): {9003145}
[2018-04-11 22:19:36] Error in run 176: Run already exists in server. Run id(s): {9003137}
[2018-04-11 22:19:36] Started run 182 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:36] Started run 183 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:36] Started run 184 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:39] Error in run 177: Run already exists in server. Run id(s): {9003147}
[2018-04-11 22:19:39] Started run 185 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:40] Error in run 178: Run already exists in server. Run id(s): {9003163}
[2018-04-11 22:19:40] Started run 186 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:40] Error in run 179: Run already exists in server. Run id(s): {9003162}
[2018-04-11 22:19:40] Error in run 181: Run already exists in server. Run id(s): {9003220}
[2018-04-11 22:19:40] Started run 187 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:40] Error in run 180: Run already exists in server. Run id(s): {9003197}
[2018-04-11 22:19:40] Started run 188 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:40] Started run 189 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:41] Error in run 182: Run already exists in server. Run id(s): {9003179}
[2018-04-11 22:19:41] Error in run 183: Run already exists in server. Run id(s): {9003205}
[2018-04-11 22:19:41] Started run 190 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:41] Error in run 184: Run already exists in server. Run id(s): {9003175}
[2018-04-11 22:19:41] Started run 191 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:41] Started run 192 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:43] Error in run 185: Run already exists in server. Run id(s): {9003182}
[2018-04-11 22:19:43] Started run 193 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:44] Error in run 186: Run already exists in server. Run id(s): {9003186}
[2018-04-11 22:19:44] Started run 194 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:44] Error in run 187: Run already exists in server. Run id(s): {9003210}
[2018-04-11 22:19:44] Error in run 188: Run already exists in server. Run id(s): {9003198}
[2018-04-11 22:19:44] Started run 195 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:45] Error in run 189: Run already exists in server. Run id(s): {9003204}
[2018-04-11 22:19:45] Started run 197 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:44] Started run 196 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:45] Error in run 190: Run already exists in server. Run id(s): {9003266}
[2018-04-11 22:19:45] Started run 198 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:45] Error in run 191: Run already exists in server. Run id(s): {9003223}
[2018-04-11 22:19:45] Error in run 192: Run already exists in server. Run id(s): {9003218}
[2018-04-11 22:19:45] Started run 199 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:45] Started run 200 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:48] Error in run 193: Run already exists in server. Run id(s): {9003229}
[2018-04-11 22:19:48] Started run 201 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:49] Error in run 194: Run already exists in server. Run id(s): {9003240}
[2018-04-11 22:19:49] Started run 202 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:49] Error in run 195: Run already exists in server. Run id(s): {9003242}
[2018-04-11 22:19:49] Started run 203 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:49] Error in run 197: Run already exists in server. Run id(s): {9003250}
[2018-04-11 22:19:49] Started run 204 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:49] Error in run 196: Run already exists in server. Run id(s): {9003257}
[2018-04-11 22:19:49] Started run 205 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:50] Error in run 198: Run already exists in server. Run id(s): {9003268}
[2018-04-11 22:19:50] Started run 206 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:50] Error in run 199: Run already exists in server. Run id(s): {9003263}
[2018-04-11 22:19:50] Error in run 200: Run already exists in server. Run id(s): {9003252}
[2018-04-11 22:19:50] Started run 207 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:50] Started run 208 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:52] Error in run 201: Run already exists in server. Run id(s): {9003274}
[2018-04-11 22:19:52] Started run 209 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:53] Error in run 202: Run already exists in server. Run id(s): {9003278}
[2018-04-11 22:19:53] Started run 210 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:54] Error in run 203: Run already exists in server. Run id(s): {9003302}
[2018-04-11 22:19:54] Started run 211 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:54] Error in run 204: Run already exists in server. Run id(s): {9003284}
[2018-04-11 22:19:54] Started run 212 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:54] Error in run 205: Run already exists in server. Run id(s): {9003342}
[2018-04-11 22:19:54] Started run 213 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:54] Error in run 206: Run already exists in server. Run id(s): {9003297}
[2018-04-11 22:19:54] Started run 214 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:55] Error in run 207: Run already exists in server. Run id(s): {9003291}
[2018-04-11 22:19:55] Started run 215 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:55] Error in run 208: Run already exists in server. Run id(s): {9003288}
[2018-04-11 22:19:55] Started run 216 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:57] Error in run 209: Run already exists in server. Run id(s): {9003329}
[2018-04-11 22:19:57] Started run 217 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:58] Error in run 210: Run already exists in server. Run id(s): {9003318}
[2018-04-11 22:19:58] Started run 218 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:58] Error in run 211: Run already exists in server. Run id(s): {9003307}
[2018-04-11 22:19:58] Started run 219 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:58] Error in run 212: Run already exists in server. Run id(s): {9003310}
[2018-04-11 22:19:58] Started run 220 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:59] Error in run 213: Run already exists in server. Run id(s): {9003316}
[2018-04-11 22:19:59] Started run 221 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:59] Error in run 214: Run already exists in server. Run id(s): {9003337}
[2018-04-11 22:19:59] Started run 222 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:59] Error in run 215: Run already exists in server. Run id(s): {9003359}
[2018-04-11 22:19:59] Started run 223 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:19:59] Error in run 216: Run already exists in server. Run id(s): {9003402}
[2018-04-11 22:19:59] Started run 224 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:02] Error in run 217: Run already exists in server. Run id(s): {9003351}
[2018-04-11 22:20:02] Started run 225 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:02] Error in run 218: Run already exists in server. Run id(s): {9003353}
[2018-04-11 22:20:02] Started run 226 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:03] Error in run 219: Run already exists in server. Run id(s): {9003347}
[2018-04-11 22:20:03] Started run 227 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:03] Error in run 220: Run already exists in server. Run id(s): {9003390}
[2018-04-11 22:20:03] Started run 228 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:03] Error in run 221: Run already exists in server. Run id(s): {9003395}
[2018-04-11 22:20:03] Started run 229 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:03] Error in run 222: Run already exists in server. Run id(s): {9003372}
[2018-04-11 22:20:03] Started run 230 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:04] Error in run 223: Run already exists in server. Run id(s): {9003384}
[2018-04-11 22:20:04] Started run 231 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:04] Error in run 224: Run already exists in server. Run id(s): {9003394}
[2018-04-11 22:20:04] Started run 232 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:06] Error in run 225: Run already exists in server. Run id(s): {9003412}
[2018-04-11 22:20:06] Started run 233 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:07] Error in run 227: Run already exists in server. Run id(s): {9003404}
[2018-04-11 22:20:07] Error in run 226: Run already exists in server. Run id(s): {9003403}
[2018-04-11 22:20:07] Started run 234 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:07] Started run 235 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:07] Error in run 228: Run already exists in server. Run id(s): {9003407}
[2018-04-11 22:20:07] Started run 236 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:08] Error in run 229: Run already exists in server. Run id(s): {9003427}
[2018-04-11 22:20:08] Started run 237 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:08] Error in run 230: Run already exists in server. Run id(s): {9003423}
[2018-04-11 22:20:08] Started run 238 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:08] Error in run 231: Run already exists in server. Run id(s): {9003417}
[2018-04-11 22:20:08] Started run 239 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:08] Error in run 232: Run already exists in server. Run id(s): {9003454}
[2018-04-11 22:20:08] Started run 240 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:11] Error in run 233: Run already exists in server. Run id(s): {9003430}
[2018-04-11 22:20:11] Started run 241 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:12] Error in run 234: Run already exists in server. Run id(s): {9003458}
[2018-04-11 22:20:12] Error in run 235: Run already exists in server. Run id(s): {9003433}
[2018-04-11 22:20:12] Started run 242 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:12] Error in run 236: Run already exists in server. Run id(s): {9003469}
[2018-04-11 22:20:12] Started run 243 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:12] Started run 244 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:12] Error in run 237: Run already exists in server. Run id(s): {9003466}
[2018-04-11 22:20:12] Started run 245 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:13] Error in run 238: Run already exists in server. Run id(s): {9003478}
[2018-04-11 22:20:13] Error in run 239: Run already exists in server. Run id(s): {9003456}
[2018-04-11 22:20:13] Started run 246 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:13] Started run 247 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:13] Error in run 240: Run already exists in server. Run id(s): {9003461}
[2018-04-11 22:20:13] Started run 248 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:16] Error in run 241: Run already exists in server. Run id(s): {9003514}
[2018-04-11 22:20:16] Started run 249 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:17] Error in run 243: Run already exists in server. Run id(s): {9003534}
[2018-04-11 22:20:17] Error in run 242: Run already exists in server. Run id(s): {9003494}
[2018-04-11 22:20:17] Started run 250 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:17] Error in run 244: Run already exists in server. Run id(s): {9003513}
[2018-04-11 22:20:17] Started run 251 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:17] Started run 252 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:17] Error in run 245: Run already exists in server. Run id(s): {9003535}
[2018-04-11 22:20:17] Started run 253 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:17] Error in run 246: Run already exists in server. Run id(s): {9003501}
[2018-04-11 22:20:17] Error in run 247: Run already exists in server. Run id(s): {9003573}
[2018-04-11 22:20:17] Started run 254 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:17] Started run 255 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:18] Error in run 248: Run already exists in server. Run id(s): {9003515}
[2018-04-11 22:20:18] Started run 256 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:20] Error in run 249: Run already exists in server. Run id(s): {9003550}
[2018-04-11 22:20:20] Started run 257 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:21] Error in run 250: Run already exists in server. Run id(s): {9003545}
[2018-04-11 22:20:21] Error in run 251: Run already exists in server. Run id(s): {9003544}
[2018-04-11 22:20:21] Started run 258 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:21] Error in run 252: Run already exists in server. Run id(s): {9003561}
[2018-04-11 22:20:21] Started run 259 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:22] Error in run 253: Run already exists in server. Run id(s): {9003553}
[2018-04-11 22:20:21] Started run 260 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:22] Started run 261 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:22] Error in run 254: Run already exists in server. Run id(s): {9003563}
[2018-04-11 22:20:22] Started run 262 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:22] Error in run 255: Run already exists in server. Run id(s): {9003585}
[2018-04-11 22:20:22] Started run 263 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:22] Error in run 256: Run already exists in server. Run id(s): {9003593}
[2018-04-11 22:20:22] Started run 264 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:25] Error in run 257: Run already exists in server. Run id(s): {9003569}
[2018-04-11 22:20:25] Started run 265 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:26] Error in run 258: Run already exists in server. Run id(s): {9003589}
[2018-04-11 22:20:26] Error in run 259: Run already exists in server. Run id(s): {9003588}
[2018-04-11 22:20:26] Error in run 260: Run already exists in server. Run id(s): {9003623}
[2018-04-11 22:20:26] Started run 266 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:26] Error in run 261: Run already exists in server. Run id(s): {9003587}
[2018-04-11 22:20:26] Started run 267 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:26] Started run 268 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:26] Started run 269 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:27] Error in run 262: Run already exists in server. Run id(s): {9003607}
[2018-04-11 22:20:27] Started run 270 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:27] Error in run 263: Run already exists in server. Run id(s): {9003619}
[2018-04-11 22:20:27] Error in run 264: Run already exists in server. Run id(s): {9003643}
[2018-04-11 22:20:27] Started run 271 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:27] Started run 272 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:30] Error in run 265: Run already exists in server. Run id(s): {9003636}
[2018-04-11 22:20:30] Started run 273 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:31] Error in run 266: Run already exists in server. Run id(s): {9003617}
[2018-04-11 22:20:31] Error in run 267: Run already exists in server. Run id(s): {9003647}
[2018-04-11 22:20:31] Started run 274 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:31] Error in run 268: Run already exists in server. Run id(s): {9003621}
[2018-04-11 22:20:31] Started run 275 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:31] Error in run 269: Run already exists in server. Run id(s): {9003652}
[2018-04-11 22:20:31] Started run 276 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:31] Started run 277 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:31] Error in run 270: Run already exists in server. Run id(s): {9003656}
[2018-04-11 22:20:31] Started run 278 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:32] Error in run 271: Run already exists in server. Run id(s): {9003672}
[2018-04-11 22:20:32] Error in run 272: Run already exists in server. Run id(s): {9003657}
[2018-04-11 22:20:32] Started run 279 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:32] Started run 280 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:35] Error in run 273: Run already exists in server. Run id(s): {9003654}
[2018-04-11 22:20:35] Started run 281 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:35] Error in run 274: Run already exists in server. Run id(s): {9003694}
[2018-04-11 22:20:35] Error in run 275: Run already exists in server. Run id(s): {9003687}
[2018-04-11 22:20:35] Error in run 276: Run already exists in server. Run id(s): {9003704}
[2018-04-11 22:20:35] Started run 282 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:35] Started run 283 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:36] Error in run 277: Run already exists in server. Run id(s): {9003684}
[2018-04-11 22:20:36] Started run 285 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:36] Error in run 278: Run already exists in server. Run id(s): {9003682}
[2018-04-11 22:20:36] Started run 286 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:35] Started run 284 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:36] Error in run 279: Run already exists in server. Run id(s): {9003690}
[2018-04-11 22:20:36] Error in run 280: Run already exists in server. Run id(s): {9003732}
[2018-04-11 22:20:36] Started run 287 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:36] Started run 288 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:39] Error in run 281: Run already exists in server. Run id(s): {9003742}
[2018-04-11 22:20:39] Started run 289 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:40] Error in run 282: Run already exists in server. Run id(s): {9003702}
[2018-04-11 22:20:40] Error in run 283: Run already exists in server. Run id(s): {9003753}
[2018-04-11 22:20:40] Started run 290 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:40] Error in run 285: Run already exists in server. Run id(s): {9003756}
[2018-04-11 22:20:40] Started run 291 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:40] Error in run 286: Run already exists in server. Run id(s): {9003731}
[2018-04-11 22:20:40] Started run 292 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:40] Started run 293 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:41] Error in run 284: Run already exists in server. Run id(s): {9003739}
[2018-04-11 22:20:41] Started run 294 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:41] Error in run 287: Run already exists in server. Run id(s): {9003728}
[2018-04-11 22:20:41] Started run 295 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:41] Error in run 288: Run already exists in server. Run id(s): {9003744}
[2018-04-11 22:20:41] Started run 296 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:44] Error in run 289: Run already exists in server. Run id(s): {9003769}
[2018-04-11 22:20:44] Started run 297 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:45] Error in run 290: Run already exists in server. Run id(s): {9003791}
[2018-04-11 22:20:45] Started run 298 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:45] Error in run 291: Run already exists in server. Run id(s): {9003801}
[2018-04-11 22:20:45] Error in run 292: Run already exists in server. Run id(s): {9003786}
[2018-04-11 22:20:45] Started run 299 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:45] Started run 300 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:45] Error in run 293: Run already exists in server. Run id(s): {9003778}
[2018-04-11 22:20:45] Started run 301 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:45] Error in run 294: Run already exists in server. Run id(s): {9003797}
[2018-04-11 22:20:45] Started run 302 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:45] Error in run 295: Run already exists in server. Run id(s): {9003789}
[2018-04-11 22:20:45] Started run 303 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:46] Error in run 296: Run already exists in server. Run id(s): {9003798}
[2018-04-11 22:20:46] Started run 304 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:49] Error in run 297: Run already exists in server. Run id(s): {9003844}
[2018-04-11 22:20:49] Started run 305 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:49] Error in run 298: Run already exists in server. Run id(s): {9003803}
[2018-04-11 22:20:49] Started run 306 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:49] Error in run 299: Run already exists in server. Run id(s): {9003835}
[2018-04-11 22:20:49] Error in run 300: Run already exists in server. Run id(s): {9003843}
[2018-04-11 22:20:49] Started run 307 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:50] Error in run 301: Run already exists in server. Run id(s): {9003851}
[2018-04-11 22:20:50] Error in run 302: Run already exists in server. Run id(s): {9003845}
[2018-04-11 22:20:50] Started run 309 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:50] Started run 310 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:50] Error in run 303: Run already exists in server. Run id(s): {9003826}
[2018-04-11 22:20:50] Started run 308 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:50] Error in run 304: Run already exists in server. Run id(s): {9003836}
[2018-04-11 22:20:50] Started run 311 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:50] Started run 312 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:53] Error in run 305: Run already exists in server. Run id(s): {9003839}
[2018-04-11 22:20:53] Started run 313 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:54] Error in run 306: Run already exists in server. Run id(s): {9003875}
[2018-04-11 22:20:54] Started run 314 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:54] Error in run 307: Run already exists in server. Run id(s): {9003871}
[2018-04-11 22:20:54] Error in run 309: Run already exists in server. Run id(s): {9003863}
[2018-04-11 22:20:54] Started run 315 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:54] Started run 316 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:54] Error in run 310: Run already exists in server. Run id(s): {9003881}
[2018-04-11 22:20:54] Started run 317 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:55] Error in run 308: Run already exists in server. Run id(s): {9003896}
[2018-04-11 22:20:55] Started run 318 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:55] Error in run 311: Run already exists in server. Run id(s): {9003898}
[2018-04-11 22:20:55] Started run 319 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:55] Error in run 312: Run already exists in server. Run id(s): {9003867}
[2018-04-11 22:20:55] Started run 320 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:58] Error in run 313: Run already exists in server. Run id(s): {9003893}
[2018-04-11 22:20:58] Started run 321 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:58] Error in run 314: Run already exists in server. Run id(s): {9003938}
[2018-04-11 22:20:58] Started run 322 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:59] Error in run 315: Run already exists in server. Run id(s): {9003925}
[2018-04-11 22:20:59] Error in run 316: Run already exists in server. Run id(s): {9003935}
[2018-04-11 22:20:59] Started run 323 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:59] Error in run 317: Run already exists in server. Run id(s): {9003894}
[2018-04-11 22:20:59] Started run 324 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:59] Started run 325 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:59] Error in run 318: Run already exists in server. Run id(s): {9003928}
[2018-04-11 22:20:59] Started run 326 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:20:59] Error in run 319: Run already exists in server. Run id(s): {9003922}
[2018-04-11 22:20:59] Started run 327 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:00] Error in run 320: Run already exists in server. Run id(s): {9003975}
[2018-04-11 22:21:00] Started run 328 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:02] Error in run 321: Run already exists in server. Run id(s): {9003929}
[2018-04-11 22:21:02] Started run 329 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:03] Error in run 322: Run already exists in server. Run id(s): {9018112, 9003954}
[2018-04-11 22:21:03] Started run 330 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:03] Error in run 323: Run already exists in server. Run id(s): {9003939}
[2018-04-11 22:21:03] Started run 331 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:04] Error in run 324: Run already exists in server. Run id(s): {9003986}
[2018-04-11 22:21:04] Error in run 325: Run already exists in server. Run id(s): {9003999}
[2018-04-11 22:21:04] Started run 332 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:04] Error in run 326: Run already exists in server. Run id(s): {9003961}
[2018-04-11 22:21:04] Started run 333 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:04] Started run 334 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:04] Error in run 327: Run already exists in server. Run id(s): {9003978}
[2018-04-11 22:21:04] Started run 335 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:04] Error in run 328: Run already exists in server. Run id(s): {9003965}
[2018-04-11 22:21:04] Started run 336 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:07] Error in run 329: Run already exists in server. Run id(s): {9003970}
[2018-04-11 22:21:07] Started run 337 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:08] Error in run 330: Run already exists in server. Run id(s): {9003980}
[2018-04-11 22:21:08] Started run 338 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:08] Error in run 331: Run already exists in server. Run id(s): {9004012}
[2018-04-11 22:21:08] Started run 339 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:08] Error in run 332: Run already exists in server. Run id(s): {9004027}
[2018-04-11 22:21:08] Error in run 333: Run already exists in server. Run id(s): {9004005}
[2018-04-11 22:21:08] Started run 340 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:08] Error in run 334: Run already exists in server. Run id(s): {9003997}
[2018-04-11 22:21:08] Started run 341 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:08] Started run 342 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:09] Error in run 335: Run already exists in server. Run id(s): {9004019}
[2018-04-11 22:21:09] Started run 343 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:09] Error in run 336: Run already exists in server. Run id(s): {9004047}
[2018-04-11 22:21:09] Started run 344 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:11] Error in run 337: Run already exists in server. Run id(s): {9004015}
[2018-04-11 22:21:11] Started run 345 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:12] Error in run 338: Run already exists in server. Run id(s): {9004054}
[2018-04-11 22:21:12] Started run 346 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:13] Error in run 339: Run already exists in server. Run id(s): {9004056}
[2018-04-11 22:21:13] Started run 347 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:13] Error in run 340: Run already exists in server. Run id(s): {9004037}
[2018-04-11 22:21:13] Started run 348 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:13] Error in run 341: Run already exists in server. Run id(s): {9004049}
[2018-04-11 22:21:13] Error in run 342: Run already exists in server. Run id(s): {9004074}
[2018-04-11 22:21:13] Started run 349 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:13] Error in run 343: Run already exists in server. Run id(s): {9004063}
[2018-04-11 22:21:13] Started run 350 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:13] Started run 351 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:14] Error in run 344: Run already exists in server. Run id(s): {9004087}
[2018-04-11 22:21:14] Started run 352 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:16] Error in run 345: Run already exists in server. Run id(s): {9004113}
[2018-04-11 22:21:16] Started run 353 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:17] Error in run 346: Run already exists in server. Run id(s): {9004097}
[2018-04-11 22:21:17] Started run 354 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:17] Error in run 347: Run already exists in server. Run id(s): {9004071}
[2018-04-11 22:21:17] Started run 355 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:17] Error in run 348: Run already exists in server. Run id(s): {9004085}
[2018-04-11 22:21:17] Started run 356 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:18] Error in run 349: Run already exists in server. Run id(s): {9004106}
[2018-04-11 22:21:18] Error in run 350: Run already exists in server. Run id(s): {9004110}
[2018-04-11 22:21:18] Started run 357 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:18] Error in run 351: Run already exists in server. Run id(s): {9004103}
[2018-04-11 22:21:18] Started run 358 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:18] Started run 359 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:18] Error in run 352: Run already exists in server. Run id(s): {9004102}
[2018-04-11 22:21:18] Started run 360 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:21] Error in run 353: Run already exists in server. Run id(s): {9004123}
[2018-04-11 22:21:21] Started run 361 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:21] Error in run 354: Run already exists in server. Run id(s): {9004129}
[2018-04-11 22:21:21] Started run 362 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:22] Error in run 355: Run already exists in server. Run id(s): {9004137}
[2018-04-11 22:21:22] Started run 363 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:22] Error in run 356: Run already exists in server. Run id(s): {9004141}
[2018-04-11 22:21:22] Started run 364 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:22] Error in run 358: Run already exists in server. Run id(s): {9004142}
[2018-04-11 22:21:22] Error in run 357: Run already exists in server. Run id(s): {9004150}
[2018-04-11 22:21:22] Error in run 359: Run already exists in server. Run id(s): {9004145}
[2018-04-11 22:21:22] Started run 365 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:22] Started run 367 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:22] Started run 366 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:23] Error in run 360: Run already exists in server. Run id(s): {9004162}
[2018-04-11 22:21:23] Started run 368 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:26] Error in run 361: Run already exists in server. Run id(s): {9004158}
[2018-04-11 22:21:26] Started run 369 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:26] Error in run 362: Run already exists in server. Run id(s): {9004167}
[2018-04-11 22:21:26] Started run 370 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:26] Error in run 363: Run already exists in server. Run id(s): {9004185}
[2018-04-11 22:21:26] Started run 371 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:27] Error in run 364: Run already exists in server. Run id(s): {9004181}
[2018-04-11 22:21:27] Started run 372 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:27] Error in run 365: Run already exists in server. Run id(s): {9004213}
[2018-04-11 22:21:27] Error in run 367: Run already exists in server. Run id(s): {9004188}
[2018-04-11 22:21:27] Started run 373 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:27] Error in run 366: Run already exists in server. Run id(s): {9004171}
[2018-04-11 22:21:27] Started run 374 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:27] Started run 375 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:27] Error in run 368: Run already exists in server. Run id(s): {9004223}
[2018-04-11 22:21:27] Started run 376 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:30] Error in run 369: Run already exists in server. Run id(s): {9004183}
[2018-04-11 22:21:30] Started run 377 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:31] Error in run 370: Run already exists in server. Run id(s): {9004195}
[2018-04-11 22:21:31] Started run 378 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:31] Error in run 371: Run already exists in server. Run id(s): {9004218}
[2018-04-11 22:21:31] Started run 379 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:31] Error in run 372: Run already exists in server. Run id(s): {9004209}
[2018-04-11 22:21:31] Started run 380 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:32] Error in run 373: Run already exists in server. Run id(s): {9004236}
[2018-04-11 22:21:32] Error in run 374: Run already exists in server. Run id(s): {9004216}
[2018-04-11 22:21:32] Error in run 375: Run already exists in server. Run id(s): {9004258}
[2018-04-11 22:21:32] Started run 381 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:32] Error in run 376: Run already exists in server. Run id(s): {9004244}
[2018-04-11 22:21:32] Started run 382 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:32] Started run 384 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:32] Started run 383 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:35] Error in run 377: Run already exists in server. Run id(s): {9004239}
[2018-04-11 22:21:35] Started run 385 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:35] Error in run 378: Run already exists in server. Run id(s): {9004249}
[2018-04-11 22:21:35] Started run 386 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:36] Error in run 379: Run already exists in server. Run id(s): {9004252}
[2018-04-11 22:21:36] Started run 387 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:36] Error in run 380: Run already exists in server. Run id(s): {9004248}
[2018-04-11 22:21:36] Started run 388 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:36] Error in run 381: Run already exists in server. Run id(s): {9004256}
[2018-04-11 22:21:36] Error in run 382: Run already exists in server. Run id(s): {9004278}
[2018-04-11 22:21:36] Started run 389 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:37] Error in run 384: Run already exists in server. Run id(s): {9004280}
[2018-04-11 22:21:36] Started run 390 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:37] Error in run 383: Run already exists in server. Run id(s): {9004270}
[2018-04-11 22:21:37] Started run 392 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:37] Started run 391 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:40] Error in run 385: Run already exists in server. Run id(s): {9004284}
[2018-04-11 22:21:40] Started run 393 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:40] Error in run 386: Run already exists in server. Run id(s): {9004305}
[2018-04-11 22:21:40] Started run 394 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:40] Error in run 387: Run already exists in server. Run id(s): {9004306}
[2018-04-11 22:21:40] Error in run 388: Run already exists in server. Run id(s): {9004282}
[2018-04-11 22:21:40] Started run 395 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:40] Started run 396 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:41] Error in run 389: Run already exists in server. Run id(s): {9004297}
[2018-04-11 22:21:41] Error in run 390: Run already exists in server. Run id(s): {9004293}
[2018-04-11 22:21:41] Error in run 392: Run already exists in server. Run id(s): {9004316}
[2018-04-11 22:21:41] Started run 397 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:41] Error in run 391: Run already exists in server. Run id(s): {9004349}
[2018-04-11 22:21:41] Started run 398 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:41] Started run 400 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:41] Started run 399 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:44] Error in run 393: Run already exists in server. Run id(s): {9004311}
[2018-04-11 22:21:44] Error in run 394: Run already exists in server. Run id(s): {9004324}
[2018-04-11 22:21:44] Started run 401 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:44] Started run 402 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:45] Error in run 395: Run already exists in server. Run id(s): {9004326}
[2018-04-11 22:21:45] Started run 403 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:45] Error in run 396: Run already exists in server. Run id(s): {9004356}
[2018-04-11 22:21:45] Started run 404 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:46] Error in run 397: Run already exists in server. Run id(s): {9004337}
[2018-04-11 22:21:46] Error in run 398: Run already exists in server. Run id(s): {9004345}
[2018-04-11 22:21:46] Started run 405 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:46] Error in run 400: Run already exists in server. Run id(s): {9004369}
[2018-04-11 22:21:46] Started run 406 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:46] Error in run 399: Run already exists in server. Run id(s): {9004383}
[2018-04-11 22:21:46] Started run 407 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:46] Started run 408 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:49] Error in run 401: Run already exists in server. Run id(s): {9004373}
[2018-04-11 22:21:49] Error in run 402: Run already exists in server. Run id(s): {9004363}
[2018-04-11 22:21:49] Started run 409 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:49] Started run 410 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:50] Error in run 403: Run already exists in server. Run id(s): {9004409}
[2018-04-11 22:21:50] Started run 411 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:50] Error in run 404: Run already exists in server. Run id(s): {9004377}
[2018-04-11 22:21:50] Started run 412 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:51] Error in run 405: Run already exists in server. Run id(s): {9004416}
[2018-04-11 22:21:51] Error in run 406: Run already exists in server. Run id(s): {9004403}
[2018-04-11 22:21:51] Error in run 407: Run already exists in server. Run id(s): {9004401}
[2018-04-11 22:21:51] Started run 413 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:51] Error in run 408: Run already exists in server. Run id(s): {9004420}
[2018-04-11 22:21:51] Started run 415 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:51] Started run 416 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:51] Started run 414 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:54] Error in run 410: Run already exists in server. Run id(s): {9004411}
[2018-04-11 22:21:54] Error in run 409: Run already exists in server. Run id(s): {9004395}
[2018-04-11 22:21:54] Started run 417 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:54] Started run 418 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:54] Error in run 411: Run already exists in server. Run id(s): {9004431}
[2018-04-11 22:21:54] Started run 419 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:54] Error in run 412: Run already exists in server. Run id(s): {9004429}
[2018-04-11 22:21:54] Started run 420 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:55] Error in run 413: Run already exists in server. Run id(s): {9004468}
[2018-04-11 22:21:55] Error in run 415: Run already exists in server. Run id(s): {9004443}
[2018-04-11 22:21:55] Started run 421 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:55] Error in run 416: Run already exists in server. Run id(s): {9004448}
[2018-04-11 22:21:55] Started run 422 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:55] Started run 423 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:56] Error in run 414: Run already exists in server. Run id(s): {9004450}
[2018-04-11 22:21:56] Started run 424 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:58] Error in run 417: Run already exists in server. Run id(s): {9004461}
[2018-04-11 22:21:58] Error in run 418: Run already exists in server. Run id(s): {9004479}
[2018-04-11 22:21:58] Started run 425 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:58] Started run 426 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:59] Error in run 419: Run already exists in server. Run id(s): {9004480}
[2018-04-11 22:21:59] Started run 427 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:21:59] Error in run 420: Run already exists in server. Run id(s): {9004467}
[2018-04-11 22:21:59] Started run 428 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:00] Error in run 422: Run already exists in server. Run id(s): {9004507}
[2018-04-11 22:22:00] Error in run 421: Run already exists in server. Run id(s): {9004490}
[2018-04-11 22:22:00] Error in run 423: Run already exists in server. Run id(s): {9004503}
[2018-04-11 22:22:00] Started run 429 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:00] Error in run 424: Run already exists in server. Run id(s): {9004500}
[2018-04-11 22:22:00] Started run 431 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:00] Started run 430 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:00] Started run 432 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:03] Error in run 425: Run already exists in server. Run id(s): {9004494}
[2018-04-11 22:22:03] Error in run 426: Run already exists in server. Run id(s): {9004505}
[2018-04-11 22:22:03] Started run 433 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:03] Started run 434 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:03] Error in run 427: Run already exists in server. Run id(s): {9004526}
[2018-04-11 22:22:03] Started run 435 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:04] Error in run 428: Run already exists in server. Run id(s): {9004542}
[2018-04-11 22:22:04] Started run 436 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:05] Error in run 429: Run already exists in server. Run id(s): {9004551}
[2018-04-11 22:22:05] Started run 437 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:05] Error in run 431: Run already exists in server. Run id(s): {9004535}
[2018-04-11 22:22:05] Started run 438 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:05] Error in run 430: Run already exists in server. Run id(s): {9004546}
[2018-04-11 22:22:05] Started run 439 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:05] Error in run 432: Run already exists in server. Run id(s): {9004547}
[2018-04-11 22:22:05] Started run 440 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:08] Error in run 433: Run already exists in server. Run id(s): {9004558}
[2018-04-11 22:22:08] Error in run 434: Run already exists in server. Run id(s): {9004543}
[2018-04-11 22:22:08] Started run 441 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:08] Started run 442 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:08] Error in run 435: Run already exists in server. Run id(s): {9004582}
[2018-04-11 22:22:08] Started run 443 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:09] Error in run 436: Run already exists in server. Run id(s): {9004560}
[2018-04-11 22:22:09] Started run 444 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:09] Error in run 437: Run already exists in server. Run id(s): {9004606}
[2018-04-11 22:22:09] Error in run 438: Run already exists in server. Run id(s): {9004588}
[2018-04-11 22:22:09] Started run 445 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:09] Error in run 439: Run already exists in server. Run id(s): {9004622}
[2018-04-11 22:22:09] Started run 447 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:10] Error in run 440: Run already exists in server. Run id(s): {9004574}
[2018-04-11 22:22:09] Started run 446 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:10] Started run 448 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:12] Error in run 442: Run already exists in server. Run id(s): {9004601}
[2018-04-11 22:22:12] Error in run 441: Run already exists in server. Run id(s): {9004583}
[2018-04-11 22:22:12] Started run 449 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:12] Started run 450 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:13] Error in run 443: Run already exists in server. Run id(s): {9004598}
[2018-04-11 22:22:13] Started run 451 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:13] Error in run 444: Run already exists in server. Run id(s): {9004614}
[2018-04-11 22:22:13] Started run 452 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:14] Error in run 445: Run already exists in server. Run id(s): {9004629}
[2018-04-11 22:22:14] Error in run 447: Run already exists in server. Run id(s): {9004616}
[2018-04-11 22:22:14] Started run 453 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:14] Error in run 446: Run already exists in server. Run id(s): {9004604}
[2018-04-11 22:22:14] Started run 454 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:14] Error in run 448: Run already exists in server. Run id(s): {9004633}
[2018-04-11 22:22:14] Started run 455 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:14] Started run 456 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:17] Error in run 449: Run already exists in server. Run id(s): {9004626}
[2018-04-11 22:22:17] Error in run 450: Run already exists in server. Run id(s): {9004639}
[2018-04-11 22:22:17] Started run 457 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:17] Started run 458 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:17] Error in run 451: Run already exists in server. Run id(s): {9004631}
[2018-04-11 22:22:17] Started run 459 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:18] Error in run 452: Run already exists in server. Run id(s): {9004675}
[2018-04-11 22:22:18] Started run 460 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:19] Error in run 453: Run already exists in server. Run id(s): {9004669}
[2018-04-11 22:22:19] Error in run 454: Run already exists in server. Run id(s): {9004658}
[2018-04-11 22:22:19] Error in run 455: Run already exists in server. Run id(s): {9004660}
[2018-04-11 22:22:19] Started run 461 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:19] Error in run 456: Run already exists in server. Run id(s): {9004659}
[2018-04-11 22:22:19] Started run 462 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:19] Started run 464 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:19] Started run 463 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:22] Error in run 457: Run already exists in server. Run id(s): {9004705}
[2018-04-11 22:22:22] Error in run 458: Run already exists in server. Run id(s): {9004672}
[2018-04-11 22:22:22] Error in run 459: Run already exists in server. Run id(s): {9004677}
[2018-04-11 22:22:22] Started run 465 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:22] Started run 466 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:22] Started run 467 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:23] Error in run 460: Run already exists in server. Run id(s): {9004671}
[2018-04-11 22:22:23] Started run 468 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:23] Error in run 461: Run already exists in server. Run id(s): {9004732}
[2018-04-11 22:22:23] Error in run 462: Run already exists in server. Run id(s): {9004713}
[2018-04-11 22:22:23] Started run 469 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:24] Error in run 464: Run already exists in server. Run id(s): {9004683}
[2018-04-11 22:22:23] Started run 470 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:24] Started run 471 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:24] Error in run 463: Run already exists in server. Run id(s): {9004707}
[2018-04-11 22:22:24] Started run 472 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:26] Error in run 465: Run already exists in server. Run id(s): {9004736}
[2018-04-11 22:22:27] Error in run 466: Run already exists in server. Run id(s): {9004689}
[2018-04-11 22:22:26] Started run 473 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:27] Error in run 467: Run already exists in server. Run id(s): {9004702}
[2018-04-11 22:22:27] Started run 474 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:27] Started run 475 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:27] Error in run 468: Run already exists in server. Run id(s): {9004716}
[2018-04-11 22:22:27] Started run 476 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:28] Error in run 469: Run already exists in server. Run id(s): {9004721}
[2018-04-11 22:22:28] Error in run 470: Run already exists in server. Run id(s): {9004754}
[2018-04-11 22:22:28] Started run 477 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:28] Error in run 471: Run already exists in server. Run id(s): {9004755}
[2018-04-11 22:22:28] Error in run 472: Run already exists in server. Run id(s): {9004741}
[2018-04-11 22:22:28] Started run 479 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:28] Started run 480 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:28] Started run 478 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:31] Error in run 473: Run already exists in server. Run id(s): {9004774}
[2018-04-11 22:22:31] Error in run 474: Run already exists in server. Run id(s): {9004747}
[2018-04-11 22:22:31] Started run 481 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:31] Error in run 475: Run already exists in server. Run id(s): {9004779}
[2018-04-11 22:22:31] Started run 482 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:31] Started run 483 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:32] Error in run 476: Run already exists in server. Run id(s): {9004796}
[2018-04-11 22:22:32] Started run 484 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:33] Error in run 477: Run already exists in server. Run id(s): {9004758}
[2018-04-11 22:22:33] Error in run 479: Run already exists in server. Run id(s): {9004769}
[2018-04-11 22:22:33] Started run 485 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:33] Error in run 480: Run already exists in server. Run id(s): {9004794}
[2018-04-11 22:22:33] Started run 486 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:33] Started run 487 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:33] Error in run 478: Run already exists in server. Run id(s): {9004832}
[2018-04-11 22:22:33] Started run 488 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:36] Error in run 481: Run already exists in server. Run id(s): {9004790}
[2018-04-11 22:22:36] Started run 489 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:36] Error in run 482: Run already exists in server. Run id(s): {9004775}
[2018-04-11 22:22:36] Started run 490 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:36] Error in run 483: Run already exists in server. Run id(s): {9004808}
[2018-04-11 22:22:36] Started run 491 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:37] Error in run 484: Run already exists in server. Run id(s): {9004816}
[2018-04-11 22:22:37] Started run 492 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:37] Error in run 485: Run already exists in server. Run id(s): {9004806}
[2018-04-11 22:22:37] Started run 493 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:37] Error in run 486: Run already exists in server. Run id(s): {9004850}
[2018-04-11 22:22:37] Started run 494 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:38] Error in run 487: Run already exists in server. Run id(s): {9004839}
[2018-04-11 22:22:38] Started run 495 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:38] Error in run 488: Run already exists in server. Run id(s): {9004836}
[2018-04-11 22:22:38] Started run 496 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:41] Error in run 490: Run already exists in server. Run id(s): {9004846}
[2018-04-11 22:22:41] Error in run 489: Run already exists in server. Run id(s): {9004824}
[2018-04-11 22:22:41] Error in run 491: Run already exists in server. Run id(s): {9004835}
[2018-04-11 22:22:41] Started run 497 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:41] Started run 498 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:41] Started run 499 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:41] Error in run 492: Run already exists in server. Run id(s): {9004868}
[2018-04-11 22:22:41] Started run 500 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:42] Error in run 493: Run already exists in server. Run id(s): {9004878}
[2018-04-11 22:22:42] Started run 501 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:42] Error in run 494: Run already exists in server. Run id(s): {9004872}
[2018-04-11 22:22:42] Error in run 495: Run already exists in server. Run id(s): {9004852}
[2018-04-11 22:22:42] Started run 502 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:42] Error in run 496: Run already exists in server. Run id(s): {9004913}
[2018-04-11 22:22:42] Started run 503 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:42] Started run 504 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:45] Error in run 497: Run already exists in server. Run id(s): {9004859}
[2018-04-11 22:22:45] Error in run 498: Run already exists in server. Run id(s): {9004866}
[2018-04-11 22:22:45] Started run 505 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:45] Error in run 499: Run already exists in server. Run id(s): {9004915}
[2018-04-11 22:22:45] Started run 506 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:45] Started run 507 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:46] Error in run 500: Run already exists in server. Run id(s): {9004887}
[2018-04-11 22:22:46] Started run 508 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:47] Error in run 501: Run already exists in server. Run id(s): {9004917}
[2018-04-11 22:22:47] Started run 509 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:47] Error in run 502: Run already exists in server. Run id(s): {9004914}
[2018-04-11 22:22:47] Error in run 503: Run already exists in server. Run id(s): {9004910}
[2018-04-11 22:22:47] Error in run 504: Run already exists in server. Run id(s): {9004900}
[2018-04-11 22:22:47] Started run 510 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:47] Started run 512 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:47] Started run 511 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:50] Error in run 505: Run already exists in server. Run id(s): {9004906}
[2018-04-11 22:22:50] Started run 513 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:50] Error in run 506: Run already exists in server. Run id(s): {9004919}
[2018-04-11 22:22:50] Error in run 507: Run already exists in server. Run id(s): {9004939}
[2018-04-11 22:22:50] Started run 514 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:50] Started run 515 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:50] Error in run 508: Run already exists in server. Run id(s): {9004936}
[2018-04-11 22:22:50] Started run 516 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:51] Error in run 509: Run already exists in server. Run id(s): {9004988}
[2018-04-11 22:22:51] Started run 517 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:52] Error in run 510: Run already exists in server. Run id(s): {9004952}
[2018-04-11 22:22:52] Error in run 512: Run already exists in server. Run id(s): {9004978}
[2018-04-11 22:22:52] Started run 518 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:52] Error in run 511: Run already exists in server. Run id(s): {9004949}
[2018-04-11 22:22:52] Started run 519 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:52] Started run 520 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:55] Error in run 513: Run already exists in server. Run id(s): {9004970}
[2018-04-11 22:22:55] Started run 521 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:55] Error in run 514: Run already exists in server. Run id(s): {9004969}
[2018-04-11 22:22:55] Error in run 515: Run already exists in server. Run id(s): {9004961}
[2018-04-11 22:22:55] Started run 522 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:55] Error in run 516: Run already exists in server. Run id(s): {9004972}
[2018-04-11 22:22:55] Started run 523 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:55] Started run 524 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:56] Error in run 517: Run already exists in server. Run id(s): {9004980}
[2018-04-11 22:22:56] Started run 525 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:56] Error in run 518: Run already exists in server. Run id(s): {9004992}
[2018-04-11 22:22:56] Started run 526 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:56] Error in run 519: Run already exists in server. Run id(s): {9005027}
[2018-04-11 22:22:56] Started run 527 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:57] Error in run 520: Run already exists in server. Run id(s): {9005021}
[2018-04-11 22:22:57] Started run 528 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:59] Error in run 521: Run already exists in server. Run id(s): {9005020}
[2018-04-11 22:22:59] Started run 529 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:22:59] Error in run 522: Run already exists in server. Run id(s): {9004999}
[2018-04-11 22:22:59] Started run 530 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:00] Error in run 523: Run already exists in server. Run id(s): {9005017}
[2018-04-11 22:23:00] Started run 531 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:00] Error in run 524: Run already exists in server. Run id(s): {9005041}
[2018-04-11 22:23:00] Started run 532 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:01] Error in run 525: Run already exists in server. Run id(s): {9005006}
[2018-04-11 22:23:01] Started run 533 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:01] Error in run 526: Run already exists in server. Run id(s): {9005024}
[2018-04-11 22:23:01] Started run 534 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:01] Error in run 527: Run already exists in server. Run id(s): {9005069}
[2018-04-11 22:23:01] Started run 535 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:01] Error in run 528: Run already exists in server. Run id(s): {9005031}
[2018-04-11 22:23:01] Started run 536 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:04] Error in run 529: Run already exists in server. Run id(s): {9005059}
[2018-04-11 22:23:04] Error in run 530: Run already exists in server. Run id(s): {9005054}
[2018-04-11 22:23:04] Started run 537 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:04] Error in run 531: Run already exists in server. Run id(s): {9005072}
[2018-04-11 22:23:04] Started run 538 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:04] Error in run 532: Run already exists in server. Run id(s): {9005064}
[2018-04-11 22:23:04] Started run 539 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:04] Started run 540 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:05] Error in run 533: Run already exists in server. Run id(s): {9005084}
[2018-04-11 22:23:05] Started run 541 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:06] Error in run 534: Run already exists in server. Run id(s): {9005057}
[2018-04-11 22:23:06] Error in run 535: Run already exists in server. Run id(s): {9005109}
[2018-04-11 22:23:06] Started run 542 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:06] Error in run 536: Run already exists in server. Run id(s): {9005075}
[2018-04-11 22:23:06] Started run 543 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:06] Started run 544 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:09] Error in run 538: Run already exists in server. Run id(s): {9005093}
[2018-04-11 22:23:09] Error in run 537: Run already exists in server. Run id(s): {9005092}
[2018-04-11 22:23:09] Error in run 539: Run already exists in server. Run id(s): {9005129}
[2018-04-11 22:23:09] Started run 545 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:09] Error in run 540: Run already exists in server. Run id(s): {9005107}
[2018-04-11 22:23:09] Started run 546 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:09] Started run 548 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:09] Started run 547 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:10] Error in run 541: Run already exists in server. Run id(s): {9005096}
[2018-04-11 22:23:10] Started run 549 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:10] Error in run 542: Run already exists in server. Run id(s): {9005110}
[2018-04-11 22:23:10] Started run 550 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:10] Error in run 543: Run already exists in server. Run id(s): {9005138}
[2018-04-11 22:23:10] Started run 551 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:11] Error in run 544: Run already exists in server. Run id(s): {9005114}
[2018-04-11 22:23:11] Started run 552 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:13] Error in run 545: Run already exists in server. Run id(s): {9005134}
[2018-04-11 22:23:13] Error in run 546: Run already exists in server. Run id(s): {9005144}
[2018-04-11 22:23:13] Started run 553 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:14] Error in run 548: Run already exists in server. Run id(s): {9005160}
[2018-04-11 22:23:13] Started run 554 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:14] Error in run 547: Run already exists in server. Run id(s): {9005158}
[2018-04-11 22:23:14] Started run 555 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:14] Started run 556 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:14] Error in run 549: Run already exists in server. Run id(s): {9005131}
[2018-04-11 22:23:14] Started run 557 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:15] Error in run 550: Run already exists in server. Run id(s): {9005152}
[2018-04-11 22:23:15] Started run 558 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:15] Error in run 551: Run already exists in server. Run id(s): {9005166}
[2018-04-11 22:23:15] Error in run 552: Run already exists in server. Run id(s): {9005155}
[2018-04-11 22:23:15] Started run 559 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:15] Started run 560 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:18] Error in run 553: Run already exists in server. Run id(s): {9005163}
[2018-04-11 22:23:18] Started run 561 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:18] Error in run 554: Run already exists in server. Run id(s): {9005177}
[2018-04-11 22:23:18] Started run 562 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:18] Error in run 555: Run already exists in server. Run id(s): {9005199}
[2018-04-11 22:23:18] Started run 563 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:18] Error in run 556: Run already exists in server. Run id(s): {9005183}
[2018-04-11 22:23:18] Started run 564 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:19] Error in run 557: Run already exists in server. Run id(s): {9005185}
[2018-04-11 22:23:19] Started run 565 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:20] Error in run 558: Run already exists in server. Run id(s): {9005231}
[2018-04-11 22:23:20] Started run 566 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:20] Error in run 559: Run already exists in server. Run id(s): {9005181}
[2018-04-11 22:23:20] Error in run 560: Run already exists in server. Run id(s): {9005207}
[2018-04-11 22:23:20] Started run 567 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:20] Started run 568 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:23] Error in run 561: Run already exists in server. Run id(s): {9005195}
[2018-04-11 22:23:23] Started run 569 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:23] Error in run 562: Run already exists in server. Run id(s): {9005257}
[2018-04-11 22:23:23] Error in run 563: Run already exists in server. Run id(s): {9005216}
[2018-04-11 22:23:23] Started run 570 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:23] Error in run 564: Run already exists in server. Run id(s): {9005219}
[2018-04-11 22:23:23] Started run 571 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:23] Started run 572 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:24] Error in run 565: Run already exists in server. Run id(s): {9005205}
[2018-04-11 22:23:24] Started run 573 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:24] Error in run 566: Run already exists in server. Run id(s): {9005226}
[2018-04-11 22:23:24] Started run 574 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:24] Error in run 568: Run already exists in server. Run id(s): {9005262}
[2018-04-11 22:23:24] Error in run 567: Run already exists in server. Run id(s): {9005241}
[2018-04-11 22:23:24] Started run 575 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:24] Started run 576 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:27] Error in run 569: Run already exists in server. Run id(s): {9005280}
[2018-04-11 22:23:27] Started run 577 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:28] Error in run 571: Run already exists in server. Run id(s): {9005282}
[2018-04-11 22:23:28] Error in run 570: Run already exists in server. Run id(s): {9005256}
[2018-04-11 22:23:28] Started run 578 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:28] Error in run 572: Run already exists in server. Run id(s): {9005245}
[2018-04-11 22:23:28] Started run 579 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:28] Started run 580 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:28] Error in run 573: Run already exists in server. Run id(s): {9005265}
[2018-04-11 22:23:28] Started run 581 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:29] Error in run 574: Run already exists in server. Run id(s): {9005277}
[2018-04-11 22:23:29] Started run 582 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:29] Error in run 575: Run already exists in server. Run id(s): {9005284}
[2018-04-11 22:23:29] Started run 583 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:29] Error in run 576: Run already exists in server. Run id(s): {9005276}
[2018-04-11 22:23:29] Started run 584 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:32] Error in run 577: Run already exists in server. Run id(s): {9005291}
[2018-04-11 22:23:32] Started run 585 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:32] Error in run 578: Run already exists in server. Run id(s): {9005321}
[2018-04-11 22:23:32] Error in run 579: Run already exists in server. Run id(s): {9005343}
[2018-04-11 22:23:32] Started run 586 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:32] Started run 587 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:32] Error in run 580: Run already exists in server. Run id(s): {9005299}
[2018-04-11 22:23:32] Started run 588 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:33] Error in run 581: Run already exists in server. Run id(s): {9005339}
[2018-04-11 22:23:33] Started run 589 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:33] Error in run 582: Run already exists in server. Run id(s): {9005336}
[2018-04-11 22:23:33] Started run 590 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:34] Error in run 583: Run already exists in server. Run id(s): {9005333}
[2018-04-11 22:23:34] Started run 591 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:34] Error in run 584: Run already exists in server. Run id(s): {9005330}
[2018-04-11 22:23:34] Started run 592 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:36] Error in run 585: Run already exists in server. Run id(s): {9005313}
[2018-04-11 22:23:36] Started run 593 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:37] Error in run 587: Run already exists in server. Run id(s): {9005344}
[2018-04-11 22:23:37] Error in run 586: Run already exists in server. Run id(s): {9005360}
[2018-04-11 22:23:37] Error in run 588: Run already exists in server. Run id(s): {9005358}
[2018-04-11 22:23:37] Started run 594 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:37] Started run 596 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:37] Started run 595 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:38] Error in run 589: Run already exists in server. Run id(s): {9005374}
[2018-04-11 22:23:38] Started run 597 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:38] Error in run 590: Run already exists in server. Run id(s): {9005354}
[2018-04-11 22:23:38] Started run 598 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:38] Error in run 591: Run already exists in server. Run id(s): {9005362}
[2018-04-11 22:23:38] Started run 599 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:38] Error in run 592: Run already exists in server. Run id(s): {9005379}
[2018-04-11 22:23:38] Started run 600 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:41] Error in run 593: Run already exists in server. Run id(s): {9005384}
[2018-04-11 22:23:41] Started run 601 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:42] Error in run 594: Run already exists in server. Run id(s): {9005414}
[2018-04-11 22:23:42] Error in run 596: Run already exists in server. Run id(s): {9005425}
[2018-04-11 22:23:42] Started run 602 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:42] Error in run 595: Run already exists in server. Run id(s): {9005424}
[2018-04-11 22:23:42] Started run 603 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:42] Started run 604 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:42] Error in run 597: Run already exists in server. Run id(s): {9005436}
[2018-04-11 22:23:42] Started run 605 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:43] Error in run 598: Run already exists in server. Run id(s): {9005390}
[2018-04-11 22:23:43] Started run 606 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:43] Error in run 599: Run already exists in server. Run id(s): {9005400}
[2018-04-11 22:23:43] Started run 607 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:43] Error in run 600: Run already exists in server. Run id(s): {9005421}
[2018-04-11 22:23:43] Started run 608 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:46] Error in run 601: Run already exists in server. Run id(s): {9005403}
[2018-04-11 22:23:46] Started run 609 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:46] Error in run 602: Run already exists in server. Run id(s): {9005455}
[2018-04-11 22:23:46] Started run 610 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:46] Error in run 603: Run already exists in server. Run id(s): {9005443}
[2018-04-11 22:23:46] Started run 611 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:47] Error in run 604: Run already exists in server. Run id(s): {9005458}
[2018-04-11 22:23:47] Started run 612 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:47] Error in run 605: Run already exists in server. Run id(s): {9005450}
[2018-04-11 22:23:47] Started run 613 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:47] Error in run 606: Run already exists in server. Run id(s): {9005446}
[2018-04-11 22:23:47] Started run 614 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:48] Error in run 607: Run already exists in server. Run id(s): {9005491}
[2018-04-11 22:23:48] Started run 615 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:48] Error in run 608: Run already exists in server. Run id(s): {9005501}
[2018-04-11 22:23:48] Started run 616 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:50] Error in run 609: Run already exists in server. Run id(s): {9005465}
[2018-04-11 22:23:50] Started run 617 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:51] Error in run 610: Run already exists in server. Run id(s): {9005483}
[2018-04-11 22:23:51] Started run 618 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:51] Error in run 611: Run already exists in server. Run id(s): {9005473}
[2018-04-11 22:23:51] Started run 619 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:51] Error in run 612: Run already exists in server. Run id(s): {9005479}
[2018-04-11 22:23:51] Started run 620 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:52] Error in run 613: Run already exists in server. Run id(s): {9005525}
[2018-04-11 22:23:52] Started run 621 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:52] Error in run 614: Run already exists in server. Run id(s): {9005507}
[2018-04-11 22:23:52] Started run 622 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:52] Error in run 615: Run already exists in server. Run id(s): {9005515}
[2018-04-11 22:23:52] Started run 623 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:53] Error in run 616: Run already exists in server. Run id(s): {9005504}
[2018-04-11 22:23:53] Started run 624 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:55] Error in run 617: Run already exists in server. Run id(s): {9005511}
[2018-04-11 22:23:55] Started run 625 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:55] Error in run 618: Run already exists in server. Run id(s): {9005539}
[2018-04-11 22:23:55] Started run 626 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:56] Error in run 619: Run already exists in server. Run id(s): {9005563}
[2018-04-11 22:23:56] Started run 627 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:56] Error in run 620: Run already exists in server. Run id(s): {9005529}
[2018-04-11 22:23:56] Started run 628 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:56] Error in run 621: Run already exists in server. Run id(s): {9005538}
[2018-04-11 22:23:56] Started run 629 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:57] Error in run 622: Run already exists in server. Run id(s): {9005536}
[2018-04-11 22:23:57] Started run 630 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:57] Error in run 623: Run already exists in server. Run id(s): {9005548}
[2018-04-11 22:23:57] Started run 631 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:23:57] Error in run 624: Run already exists in server. Run id(s): {9005561}
[2018-04-11 22:23:57] Started run 632 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:00] Error in run 625: Run already exists in server. Run id(s): {9005549}
[2018-04-11 22:24:00] Started run 633 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:00] Error in run 626: Run already exists in server. Run id(s): {9005576}
[2018-04-11 22:24:00] Started run 634 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:00] Error in run 627: Run already exists in server. Run id(s): {9005581}
[2018-04-11 22:24:00] Started run 635 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:00] Error in run 628: Run already exists in server. Run id(s): {9005556}
[2018-04-11 22:24:00] Started run 636 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:01] Error in run 629: Run already exists in server. Run id(s): {9005585}
[2018-04-11 22:24:01] Started run 637 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:01] Error in run 630: Run already exists in server. Run id(s): {9005618}
[2018-04-11 22:24:01] Error in run 631: Run already exists in server. Run id(s): {9005570}
[2018-04-11 22:24:01] Started run 638 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:01] Started run 639 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:02] Error in run 632: Run already exists in server. Run id(s): {9005617}
[2018-04-11 22:24:02] Started run 640 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:04] Error in run 633: Run already exists in server. Run id(s): {9005583}
[2018-04-11 22:24:04] Started run 641 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:05] Error in run 634: Run already exists in server. Run id(s): {9005591}
[2018-04-11 22:24:05] Started run 642 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:05] Error in run 635: Run already exists in server. Run id(s): {9005607}
[2018-04-11 22:24:05] Started run 643 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:05] Error in run 636: Run already exists in server. Run id(s): {9005654}
[2018-04-11 22:24:05] Started run 644 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:05] Error in run 637: Run already exists in server. Run id(s): {9005622}
[2018-04-11 22:24:05] Started run 645 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:06] Error in run 638: Run already exists in server. Run id(s): {9005632}
[2018-04-11 22:24:06] Error in run 639: Run already exists in server. Run id(s): {9005629}
[2018-04-11 22:24:06] Started run 646 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:06] Started run 647 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:07] Error in run 640: Run already exists in server. Run id(s): {9005631}
[2018-04-11 22:24:07] Started run 648 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:09] Error in run 641: Run already exists in server. Run id(s): {9005650}
[2018-04-11 22:24:09] Started run 649 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:09] Error in run 642: Run already exists in server. Run id(s): {9005663}
[2018-04-11 22:24:09] Started run 650 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:10] Error in run 643: Run already exists in server. Run id(s): {9005687}
[2018-04-11 22:24:10] Started run 651 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:10] Error in run 644: Run already exists in server. Run id(s): {9005657}
[2018-04-11 22:24:10] Started run 652 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:10] Error in run 645: Run already exists in server. Run id(s): {9005683}
[2018-04-11 22:24:10] Started run 653 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:11] Error in run 647: Run already exists in server. Run id(s): {9005699}
[2018-04-11 22:24:11] Error in run 646: Run already exists in server. Run id(s): {9005682}
[2018-04-11 22:24:11] Started run 654 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:11] Started run 655 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:11] Error in run 648: Run already exists in server. Run id(s): {9005690}
[2018-04-11 22:24:11] Started run 656 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:13] Error in run 649: Run already exists in server. Run id(s): {9005684}
[2018-04-11 22:24:13] Started run 657 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:14] Error in run 650: Run already exists in server. Run id(s): {9005716}
[2018-04-11 22:24:14] Started run 658 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:14] Error in run 651: Run already exists in server. Run id(s): {9005696}
[2018-04-11 22:24:14] Started run 659 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:15] Error in run 652: Run already exists in server. Run id(s): {9005721}
[2018-04-11 22:24:15] Error in run 653: Run already exists in server. Run id(s): {9005704}
[2018-04-11 22:24:15] Started run 660 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:15] Started run 661 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:15] Error in run 654: Run already exists in server. Run id(s): {9005719}
[2018-04-11 22:24:15] Error in run 655: Run already exists in server. Run id(s): {9005707}
[2018-04-11 22:24:15] Started run 662 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:15] Started run 663 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:16] Error in run 656: Run already exists in server. Run id(s): {9005724}
[2018-04-11 22:24:16] Started run 664 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:18] Error in run 657: Run already exists in server. Run id(s): {9005735}
[2018-04-11 22:24:18] Started run 665 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:19] Error in run 658: Run already exists in server. Run id(s): {9005733}
[2018-04-11 22:24:19] Started run 666 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:19] Error in run 659: Run already exists in server. Run id(s): {9005741}
[2018-04-11 22:24:19] Started run 667 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:19] Error in run 660: Run already exists in server. Run id(s): {9005739}
[2018-04-11 22:24:19] Error in run 661: Run already exists in server. Run id(s): {9005759}
[2018-04-11 22:24:19] Started run 668 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:19] Started run 669 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:20] Error in run 662: Run already exists in server. Run id(s): {9005743}
[2018-04-11 22:24:20] Started run 670 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:20] Error in run 663: Run already exists in server. Run id(s): {9005790}
[2018-04-11 22:24:20] Started run 671 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:21] Error in run 664: Run already exists in server. Run id(s): {9005746}
[2018-04-11 22:24:21] Started run 672 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:23] Error in run 665: Run already exists in server. Run id(s): {9005770}
[2018-04-11 22:24:23] Started run 673 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:23] Error in run 666: Run already exists in server. Run id(s): {9005784}
[2018-04-11 22:24:23] Started run 674 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:24] Error in run 667: Run already exists in server. Run id(s): {9005768}
[2018-04-11 22:24:24] Started run 675 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:24] Error in run 668: Run already exists in server. Run id(s): {9005788}
[2018-04-11 22:24:24] Error in run 669: Run already exists in server. Run id(s): {9005808}
[2018-04-11 22:24:24] Started run 676 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:24] Started run 677 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:25] Error in run 670: Run already exists in server. Run id(s): {9005778}
[2018-04-11 22:24:25] Started run 678 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:25] Error in run 671: Run already exists in server. Run id(s): {9005793}
[2018-04-11 22:24:25] Started run 679 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:26] Error in run 672: Run already exists in server. Run id(s): {9005791}
[2018-04-11 22:24:26] Started run 680 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:27] Error in run 673: Run already exists in server. Run id(s): {9005812}
[2018-04-11 22:24:27] Started run 681 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:28] Error in run 674: Run already exists in server. Run id(s): {9005802}
[2018-04-11 22:24:28] Started run 682 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:28] Error in run 675: Run already exists in server. Run id(s): {9005813}
[2018-04-11 22:24:28] Started run 683 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:29] Error in run 676: Run already exists in server. Run id(s): {9005845}
[2018-04-11 22:24:29] Error in run 677: Run already exists in server. Run id(s): {9005822}
[2018-04-11 22:24:29] Started run 684 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:29] Started run 685 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:29] Error in run 678: Run already exists in server. Run id(s): {9005859}
[2018-04-11 22:24:29] Started run 686 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:30] Error in run 679: Run already exists in server. Run id(s): {9005826}
[2018-04-11 22:24:30] Started run 687 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:30] Error in run 680: Run already exists in server. Run id(s): {9005844}
[2018-04-11 22:24:30] Started run 688 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:32] Error in run 681: Run already exists in server. Run id(s): {9005864}
[2018-04-11 22:24:32] Started run 689 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:33] Error in run 682: Run already exists in server. Run id(s): {9005842}
[2018-04-11 22:24:33] Error in run 683: Run already exists in server. Run id(s): {9005853}
[2018-04-11 22:24:33] Started run 690 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:33] Started run 691 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:33] Error in run 685: Run already exists in server. Run id(s): {9005871}
[2018-04-11 22:24:33] Error in run 684: Run already exists in server. Run id(s): {9005869}
[2018-04-11 22:24:33] Started run 692 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:33] Started run 693 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:34] Error in run 686: Run already exists in server. Run id(s): {9005876}
[2018-04-11 22:24:34] Started run 694 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:34] Error in run 687: Run already exists in server. Run id(s): {9005861}
[2018-04-11 22:24:34] Started run 695 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:35] Error in run 688: Run already exists in server. Run id(s): {9005913}
[2018-04-11 22:24:35] Started run 696 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:37] Error in run 689: Run already exists in server. Run id(s): {9005878}
[2018-04-11 22:24:37] Started run 697 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:37] Error in run 690: Run already exists in server. Run id(s): {9005889}
[2018-04-11 22:24:37] Error in run 691: Run already exists in server. Run id(s): {9019610, 9019683, 9005934}
[2018-04-11 22:24:37] Started run 698 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:37] Started run 699 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:38] Error in run 692: Run already exists in server. Run id(s): {9005947}
[2018-04-11 22:24:38] Started run 700 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:38] Error in run 693: Run already exists in server. Run id(s): {9005897}
[2018-04-11 22:24:38] Started run 701 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:38] Error in run 694: Run already exists in server. Run id(s): {9005914}
[2018-04-11 22:24:38] Started run 702 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:39] Error in run 695: Run already exists in server. Run id(s): {9005907}
[2018-04-11 22:24:39] Started run 703 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:40] Error in run 696: Run already exists in server. Run id(s): {9005923}
[2018-04-11 22:24:40] Started run 704 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:41] Error in run 697: Run already exists in server. Run id(s): {9005932}
[2018-04-11 22:24:41] Started run 705 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:42] Error in run 698: Run already exists in server. Run id(s): {9005936}
[2018-04-11 22:24:42] Error in run 699: Run already exists in server. Run id(s): {9005949}
[2018-04-11 22:24:42] Started run 706 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:42] Started run 707 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:43] Error in run 700: Run already exists in server. Run id(s): {9005961}
[2018-04-11 22:24:43] Started run 708 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:43] Error in run 701: Run already exists in server. Run id(s): {9005943}
[2018-04-11 22:24:43] Started run 709 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:43] Error in run 702: Run already exists in server. Run id(s): {9005964}
[2018-04-11 22:24:43] Started run 710 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:43] Error in run 703: Run already exists in server. Run id(s): {9005958}
[2018-04-11 22:24:43] Started run 711 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:44] Error in run 704: Run already exists in server. Run id(s): {9005968}
[2018-04-11 22:24:44] Started run 712 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:46] Error in run 705: Run already exists in server. Run id(s): {9005992}
[2018-04-11 22:24:46] Started run 713 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:47] Error in run 706: Run already exists in server. Run id(s): {9005988}
[2018-04-11 22:24:47] Error in run 707: Run already exists in server. Run id(s): {9005984}
[2018-04-11 22:24:47] Started run 714 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:47] Started run 715 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:47] Error in run 708: Run already exists in server. Run id(s): {9006029}
[2018-04-11 22:24:47] Started run 716 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:48] Error in run 709: Run already exists in server. Run id(s): {9005981}
[2018-04-11 22:24:48] Started run 717 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:48] Error in run 710: Run already exists in server. Run id(s): {9005977}
[2018-04-11 22:24:48] Started run 718 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:48] Error in run 711: Run already exists in server. Run id(s): {9006001}
[2018-04-11 22:24:48] Started run 719 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:49] Error in run 712: Run already exists in server. Run id(s): {9006020}
[2018-04-11 22:24:49] Started run 720 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:51] Error in run 713: Run already exists in server. Run id(s): {9006017}
[2018-04-11 22:24:51] Started run 721 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:51] Error in run 714: Run already exists in server. Run id(s): {9006007}
[2018-04-11 22:24:51] Started run 722 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:51] Error in run 715: Run already exists in server. Run id(s): {9006045}
[2018-04-11 22:24:51] Started run 723 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:52] Error in run 716: Run already exists in server. Run id(s): {9006016}
[2018-04-11 22:24:52] Started run 724 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:52] Error in run 717: Run already exists in server. Run id(s): {9006049}
[2018-04-11 22:24:52] Started run 725 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:52] Error in run 718: Run already exists in server. Run id(s): {9006032}
[2018-04-11 22:24:52] Started run 726 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:53] Error in run 719: Run already exists in server. Run id(s): {9006030}
[2018-04-11 22:24:53] Started run 727 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:54] Error in run 720: Run already exists in server. Run id(s): {9006056}
[2018-04-11 22:24:54] Started run 728 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:55] Error in run 721: Run already exists in server. Run id(s): {9006076}
[2018-04-11 22:24:55] Started run 729 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:56] Error in run 722: Run already exists in server. Run id(s): {9006067}
[2018-04-11 22:24:56] Started run 730 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:56] Error in run 723: Run already exists in server. Run id(s): {9006081}
[2018-04-11 22:24:56] Started run 731 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:57] Error in run 724: Run already exists in server. Run id(s): {9006073}
[2018-04-11 22:24:57] Started run 732 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:57] Error in run 725: Run already exists in server. Run id(s): {9006093}
[2018-04-11 22:24:57] Error in run 726: Run already exists in server. Run id(s): {9006071}
[2018-04-11 22:24:57] Started run 733 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:57] Started run 734 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:57] Error in run 727: Run already exists in server. Run id(s): {9006084}
[2018-04-11 22:24:57] Started run 735 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:24:58] Error in run 728: Run already exists in server. Run id(s): {9006094}
[2018-04-11 22:24:58] Started run 736 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:00] Error in run 729: Run already exists in server. Run id(s): {9006126}
[2018-04-11 22:25:00] Started run 737 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:01] Error in run 730: Run already exists in server. Run id(s): {9006097}
[2018-04-11 22:25:01] Started run 738 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:01] Error in run 731: Run already exists in server. Run id(s): {9006113}
[2018-04-11 22:25:01] Started run 739 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:01] Error in run 732: Run already exists in server. Run id(s): {9006114}
[2018-04-11 22:25:01] Started run 740 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:02] Error in run 734: Run already exists in server. Run id(s): {9006158}
[2018-04-11 22:25:02] Error in run 733: Run already exists in server. Run id(s): {9006121}
[2018-04-11 22:25:02] Started run 741 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:02] Error in run 735: Run already exists in server. Run id(s): {9006156}
[2018-04-11 22:25:02] Started run 742 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:02] Started run 743 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:03] Error in run 736: Run already exists in server. Run id(s): {9006147}
[2018-04-11 22:25:03] Started run 744 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:05] Error in run 737: Run already exists in server. Run id(s): {9006133}
[2018-04-11 22:25:05] Started run 745 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:05] Error in run 738: Run already exists in server. Run id(s): {9006149}
[2018-04-11 22:25:05] Started run 746 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:06] Error in run 739: Run already exists in server. Run id(s): {9006153}
[2018-04-11 22:25:06] Started run 747 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:06] Error in run 740: Run already exists in server. Run id(s): {9006171}
[2018-04-11 22:25:06] Started run 748 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:06] Error in run 741: Run already exists in server. Run id(s): {9006146}
[2018-04-11 22:25:06] Started run 749 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:06] Error in run 742: Run already exists in server. Run id(s): {9006182}
[2018-04-11 22:25:06] Started run 750 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:06] Error in run 743: Run already exists in server. Run id(s): {9006170}
[2018-04-11 22:25:06] Started run 751 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:08] Error in run 744: Run already exists in server. Run id(s): {9006200}
[2018-04-11 22:25:08] Started run 752 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:09] Error in run 745: Run already exists in server. Run id(s): {9006212}
[2018-04-11 22:25:09] Started run 753 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:10] Error in run 746: Run already exists in server. Run id(s): {9006193}
[2018-04-11 22:25:10] Started run 754 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:10] Error in run 747: Run already exists in server. Run id(s): {9006195}
[2018-04-11 22:25:10] Started run 755 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:10] Error in run 748: Run already exists in server. Run id(s): {9006187}
[2018-04-11 22:25:10] Started run 756 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:11] Error in run 749: Run already exists in server. Run id(s): {9006189}
[2018-04-11 22:25:11] Error in run 750: Run already exists in server. Run id(s): {9006210}
[2018-04-11 22:25:11] Started run 757 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:11] Error in run 751: Run already exists in server. Run id(s): {9006214}
[2018-04-11 22:25:11] Started run 758 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:11] Started run 759 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:12] Error in run 752: Run already exists in server. Run id(s): {9006219}
[2018-04-11 22:25:12] Started run 760 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:14] Error in run 753: Run already exists in server. Run id(s): {9006239}
[2018-04-11 22:25:14] Started run 761 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:14] Error in run 754: Run already exists in server. Run id(s): {9006235}
[2018-04-11 22:25:14] Started run 762 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:15] Error in run 755: Run already exists in server. Run id(s): {9006227}
[2018-04-11 22:25:15] Started run 763 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:15] Error in run 756: Run already exists in server. Run id(s): {9006277}
[2018-04-11 22:25:15] Started run 764 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:16] Error in run 757: Run already exists in server. Run id(s): {9006276}
[2018-04-11 22:25:16] Error in run 758: Run already exists in server. Run id(s): {9006270}
[2018-04-11 22:25:16] Started run 765 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:16] Started run 766 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:16] Error in run 759: Run already exists in server. Run id(s): {9006241}
[2018-04-11 22:25:16] Started run 767 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:17] Error in run 760: Run already exists in server. Run id(s): {9006254}
[2018-04-11 22:25:17] Started run 768 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:19] Error in run 761: Run already exists in server. Run id(s): {9006259}
[2018-04-11 22:25:19] Started run 769 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:19] Error in run 762: Run already exists in server. Run id(s): {9006299}
[2018-04-11 22:25:19] Started run 770 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:20] Error in run 763: Run already exists in server. Run id(s): {9006283}
[2018-04-11 22:25:20] Error in run 764: Run already exists in server. Run id(s): {9006280}
[2018-04-11 22:25:20] Started run 771 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:20] Started run 772 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:20] Error in run 766: Run already exists in server. Run id(s): {9006333}
[2018-04-11 22:25:20] Error in run 765: Run already exists in server. Run id(s): {9006291}
[2018-04-11 22:25:20] Started run 773 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:21] Error in run 767: Run already exists in server. Run id(s): {9006348}
[2018-04-11 22:25:20] Started run 774 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:21] Started run 775 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:22] Error in run 768: Run already exists in server. Run id(s): {9006317}
[2018-04-11 22:25:22] Started run 776 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:24] Error in run 769: Run already exists in server. Run id(s): {9006303}
[2018-04-11 22:25:24] Error in run 770: Run already exists in server. Run id(s): {9006335}
[2018-04-11 22:25:24] Started run 777 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:24] Started run 778 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:24] Error in run 771: Run already exists in server. Run id(s): {9006338}
[2018-04-11 22:25:24] Error in run 772: Run already exists in server. Run id(s): {9006326}
[2018-04-11 22:25:24] Started run 779 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:24] Started run 780 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:25] Error in run 773: Run already exists in server. Run id(s): {9006320}
[2018-04-11 22:25:25] Error in run 774: Run already exists in server. Run id(s): {9006384}
[2018-04-11 22:25:25] Started run 781 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:25] Error in run 775: Run already exists in server. Run id(s): {9006389}
[2018-04-11 22:25:25] Started run 782 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:25] Started run 783 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:27] Error in run 776: Run already exists in server. Run id(s): {9006352}
[2018-04-11 22:25:27] Started run 784 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:28] Error in run 777: Run already exists in server. Run id(s): {9006373}
[2018-04-11 22:25:28] Started run 785 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:28] Error in run 778: Run already exists in server. Run id(s): {9006350}
[2018-04-11 22:25:28] Started run 786 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:29] Error in run 779: Run already exists in server. Run id(s): {9006367}
[2018-04-11 22:25:29] Error in run 780: Run already exists in server. Run id(s): {9006381}
[2018-04-11 22:25:29] Started run 787 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:29] Started run 788 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:30] Error in run 781: Run already exists in server. Run id(s): {9006380}
[2018-04-11 22:25:30] Error in run 782: Run already exists in server. Run id(s): {9006377}
[2018-04-11 22:25:30] Started run 789 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:30] Error in run 783: Run already exists in server. Run id(s): {9006432}
[2018-04-11 22:25:30] Started run 790 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:30] Started run 791 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:31] Error in run 784: Run already exists in server. Run id(s): {9006394}
[2018-04-11 22:25:31] Started run 792 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:33] Error in run 785: Run already exists in server. Run id(s): {9006414}
[2018-04-11 22:25:33] Error in run 786: Run already exists in server. Run id(s): {9006405}
[2018-04-11 22:25:33] Started run 793 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:33] Started run 794 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:34] Error in run 787: Run already exists in server. Run id(s): {9006407}
[2018-04-11 22:25:34] Error in run 788: Run already exists in server. Run id(s): {9006416}
[2018-04-11 22:25:34] Started run 795 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:34] Started run 796 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:34] Error in run 789: Run already exists in server. Run id(s): {9006411}
[2018-04-11 22:25:34] Started run 797 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:35] Error in run 790: Run already exists in server. Run id(s): {9006431}
[2018-04-11 22:25:35] Started run 798 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:35] Error in run 791: Run already exists in server. Run id(s): {9006424}
[2018-04-11 22:25:35] Started run 799 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:36] Error in run 792: Run already exists in server. Run id(s): {9006435}
[2018-04-11 22:25:36] Started run 800 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:38] Error in run 793: Run already exists in server. Run id(s): {9006427}
[2018-04-11 22:25:38] Error in run 794: Run already exists in server. Run id(s): {9006476}
[2018-04-11 22:25:38] Started run 801 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:38] Started run 802 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:38] Error in run 796: Run already exists in server. Run id(s): {9006461}
[2018-04-11 22:25:38] Error in run 795: Run already exists in server. Run id(s): {9006474}
[2018-04-11 22:25:38] Started run 803 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:38] Started run 804 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:39] Error in run 797: Run already exists in server. Run id(s): {9006477}
[2018-04-11 22:25:39] Error in run 798: Run already exists in server. Run id(s): {9006462}
[2018-04-11 22:25:39] Started run 805 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:39] Error in run 799: Run already exists in server. Run id(s): {9006459}
[2018-04-11 22:25:39] Started run 806 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:39] Started run 807 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:41] Error in run 800: Run already exists in server. Run id(s): {9006480}
[2018-04-11 22:25:41] Started run 808 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:42] Error in run 802: Run already exists in server. Run id(s): {9006478}
[2018-04-11 22:25:42] Error in run 801: Run already exists in server. Run id(s): {9006468}
[2018-04-11 22:25:42] Started run 809 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:42] Started run 810 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:43] Error in run 803: Run already exists in server. Run id(s): {9006486}
[2018-04-11 22:25:43] Error in run 804: Run already exists in server. Run id(s): {9006496}
[2018-04-11 22:25:43] Started run 811 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:43] Started run 812 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:44] Error in run 805: Run already exists in server. Run id(s): {9006493}
[2018-04-11 22:25:44] Error in run 806: Run already exists in server. Run id(s): {9006500}
[2018-04-11 22:25:44] Started run 813 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:44] Error in run 807: Run already exists in server. Run id(s): {9006494}
[2018-04-11 22:25:44] Started run 814 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:44] Started run 815 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:45] Error in run 808: Run already exists in server. Run id(s): {9006504}
[2018-04-11 22:25:45] Started run 816 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:47] Error in run 809: Run already exists in server. Run id(s): {9006511}
[2018-04-11 22:25:47] Error in run 810: Run already exists in server. Run id(s): {9006499}
[2018-04-11 22:25:47] Started run 817 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:47] Started run 818 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:48] Error in run 811: Run already exists in server. Run id(s): {9006517}
[2018-04-11 22:25:48] Error in run 812: Run already exists in server. Run id(s): {9006508}
[2018-04-11 22:25:48] Started run 819 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:48] Started run 820 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:49] Error in run 814: Run already exists in server. Run id(s): {9006530}
[2018-04-11 22:25:49] Error in run 813: Run already exists in server. Run id(s): {9006515}
[2018-04-11 22:25:49] Error in run 815: Run already exists in server. Run id(s): {9006522}
[2018-04-11 22:25:49] Started run 821 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:49] Started run 823 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:49] Started run 822 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:50] Error in run 816: Run already exists in server. Run id(s): {9006523}
[2018-04-11 22:25:50] Started run 824 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:52] Error in run 817: Run already exists in server. Run id(s): {9006524}
[2018-04-11 22:25:52] Error in run 818: Run already exists in server. Run id(s): {9006531}
[2018-04-11 22:25:52] Started run 825 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:52] Started run 826 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:52] Error in run 819: Run already exists in server. Run id(s): {9006533}
[2018-04-11 22:25:52] Error in run 820: Run already exists in server. Run id(s): {9006535}
[2018-04-11 22:25:52] Started run 827 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:52] Started run 828 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:53] Error in run 821: Run already exists in server. Run id(s): {9006537}
[2018-04-11 22:25:53] Started run 829 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:53] Error in run 823: Run already exists in server. Run id(s): {9006548}
[2018-04-11 22:25:53] Started run 830 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:53] Error in run 822: Run already exists in server. Run id(s): {9006542}
[2018-04-11 22:25:53] Started run 831 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:55] Error in run 824: Run already exists in server. Run id(s): {9006566}
[2018-04-11 22:25:55] Started run 832 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:56] Error in run 825: Run already exists in server. Run id(s): {9006563}
[2018-04-11 22:25:56] Error in run 826: Run already exists in server. Run id(s): {9006549}
[2018-04-11 22:25:56] Started run 833 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:56] Started run 834 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:57] Error in run 827: Run already exists in server. Run id(s): {9006541}
[2018-04-11 22:25:57] Error in run 828: Run already exists in server. Run id(s): {9006554}
[2018-04-11 22:25:57] Started run 835 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:57] Started run 836 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:58] Error in run 829: Run already exists in server. Run id(s): {9006558}
[2018-04-11 22:25:58] Error in run 830: Run already exists in server. Run id(s): {9006582}
[2018-04-11 22:25:58] Started run 837 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:58] Error in run 831: Run already exists in server. Run id(s): {9006592}
[2018-04-11 22:25:58] Started run 838 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:58] Started run 839 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:25:59] Error in run 832: Run already exists in server. Run id(s): {9006595}
[2018-04-11 22:25:59] Started run 840 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:01] Error in run 833: Run already exists in server. Run id(s): {9006580}
[2018-04-11 22:26:01] Error in run 834: Run already exists in server. Run id(s): {9006602}
[2018-04-11 22:26:01] Started run 841 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:01] Started run 842 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:02] Error in run 835: Run already exists in server. Run id(s): {9006581}
[2018-04-11 22:26:02] Started run 843 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:02] Error in run 836: Run already exists in server. Run id(s): {9006585}
[2018-04-11 22:26:02] Started run 844 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:03] Error in run 837: Run already exists in server. Run id(s): {9006604}
[2018-04-11 22:26:03] Started run 845 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:03] Error in run 838: Run already exists in server. Run id(s): {9006591}
[2018-04-11 22:26:03] Started run 846 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:03] Error in run 839: Run already exists in server. Run id(s): {9006599}
[2018-04-11 22:26:03] Started run 847 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:04] Error in run 840: Run already exists in server. Run id(s): {9006629}
[2018-04-11 22:26:04] Started run 848 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:06] Error in run 841: Run already exists in server. Run id(s): {9006608}
[2018-04-11 22:26:06] Started run 849 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:06] Error in run 842: Run already exists in server. Run id(s): {9006619}
[2018-04-11 22:26:06] Started run 850 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:07] Error in run 843: Run already exists in server. Run id(s): {9006605}
[2018-04-11 22:26:07] Error in run 844: Run already exists in server. Run id(s): {9006621}
[2018-04-11 22:26:07] Started run 851 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:07] Started run 852 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:07] Error in run 845: Run already exists in server. Run id(s): {9006618}
[2018-04-11 22:26:07] Error in run 846: Run already exists in server. Run id(s): {9006630}
[2018-04-11 22:26:07] Started run 853 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:07] Error in run 847: Run already exists in server. Run id(s): {9006640}
[2018-04-11 22:26:07] Started run 855 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:07] Started run 854 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:09] Error in run 848: Run already exists in server. Run id(s): {9006660}
[2018-04-11 22:26:09] Started run 856 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:10] Error in run 850: Run already exists in server. Run id(s): {9006643}
[2018-04-11 22:26:10] Error in run 849: Run already exists in server. Run id(s): {9006634}
[2018-04-11 22:26:10] Started run 857 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:10] Started run 858 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:11] Error in run 851: Run already exists in server. Run id(s): {9006631}
[2018-04-11 22:26:11] Error in run 852: Run already exists in server. Run id(s): {9006655}
[2018-04-11 22:26:11] Started run 859 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:11] Started run 860 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:12] Error in run 853: Run already exists in server. Run id(s): {9006639}
[2018-04-11 22:26:12] Started run 861 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:12] Error in run 855: Run already exists in server. Run id(s): {9006662}
[2018-04-11 22:26:12] Started run 862 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:12] Error in run 854: Run already exists in server. Run id(s): {9006647}
[2018-04-11 22:26:12] Started run 863 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:14] Error in run 856: Run already exists in server. Run id(s): {9006673}
[2018-04-11 22:26:14] Started run 864 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:15] Error in run 857: Run already exists in server. Run id(s): {9006671}
[2018-04-11 22:26:15] Started run 865 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:15] Error in run 858: Run already exists in server. Run id(s): {9006649}
[2018-04-11 22:26:15] Started run 866 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:16] Error in run 860: Run already exists in server. Run id(s): {9006675}
[2018-04-11 22:26:16] Error in run 859: Run already exists in server. Run id(s): {9006674}
[2018-04-11 22:26:16] Started run 867 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:16] Started run 868 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:17] Error in run 862: Run already exists in server. Run id(s): {9006681}
[2018-04-11 22:26:17] Error in run 861: Run already exists in server. Run id(s): {9006664}
[2018-04-11 22:26:17] Started run 869 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:17] Error in run 863: Run already exists in server. Run id(s): {9006690}
[2018-04-11 22:26:17] Started run 870 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:17] Started run 871 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:18] Error in run 864: Run already exists in server. Run id(s): {9006700}
[2018-04-11 22:26:18] Started run 872 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:20] Error in run 865: Run already exists in server. Run id(s): {9006691}
[2018-04-11 22:26:20] Error in run 866: Run already exists in server. Run id(s): {9006714}
[2018-04-11 22:26:20] Started run 873 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:20] Started run 874 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:21] Error in run 867: Run already exists in server. Run id(s): {9006688}
[2018-04-11 22:26:21] Error in run 868: Run already exists in server. Run id(s): {9006694}
[2018-04-11 22:26:21] Started run 875 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:21] Started run 876 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:21] Error in run 869: Run already exists in server. Run id(s): {9006689}
[2018-04-11 22:26:21] Error in run 870: Run already exists in server. Run id(s): {9006702}
[2018-04-11 22:26:21] Started run 877 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:21] Error in run 871: Run already exists in server. Run id(s): {9006706}
[2018-04-11 22:26:21] Started run 878 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:21] Started run 879 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:23] Error in run 872: Run already exists in server. Run id(s): {9006712}
[2018-04-11 22:26:23] Started run 880 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:24] Error in run 873: Run already exists in server. Run id(s): {9006715}
[2018-04-11 22:26:24] Started run 881 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:25] Error in run 874: Run already exists in server. Run id(s): {9006718}
[2018-04-11 22:26:25] Started run 882 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:25] Error in run 875: Run already exists in server. Run id(s): {9006719}
[2018-04-11 22:26:25] Started run 883 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:25] Error in run 876: Run already exists in server. Run id(s): {9006720}
[2018-04-11 22:26:25] Started run 884 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:26] Error in run 877: Run already exists in server. Run id(s): {9006729}
[2018-04-11 22:26:26] Started run 885 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:26] Error in run 878: Run already exists in server. Run id(s): {9006731}
[2018-04-11 22:26:26] Started run 886 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:26] Error in run 879: Run already exists in server. Run id(s): {9006736}
[2018-04-11 22:26:26] Started run 887 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:28] Error in run 880: Run already exists in server. Run id(s): {9006747}
[2018-04-11 22:26:28] Started run 888 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:29] Error in run 881: Run already exists in server. Run id(s): {9006735}
[2018-04-11 22:26:29] Error in run 882: Run already exists in server. Run id(s): {9006732}
[2018-04-11 22:26:29] Started run 889 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:29] Started run 890 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:30] Error in run 883: Run already exists in server. Run id(s): {9006752}
[2018-04-11 22:26:30] Started run 891 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:30] Error in run 884: Run already exists in server. Run id(s): {9006741}
[2018-04-11 22:26:30] Started run 892 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:31] Error in run 885: Run already exists in server. Run id(s): {9006760}
[2018-04-11 22:26:31] Started run 893 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:31] Error in run 886: Run already exists in server. Run id(s): {9006743}
[2018-04-11 22:26:31] Started run 894 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:31] Error in run 887: Run already exists in server. Run id(s): {9006745}
[2018-04-11 22:26:31] Started run 895 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:32] Error in run 888: Run already exists in server. Run id(s): {9006746}
[2018-04-11 22:26:32] Started run 896 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:34] Error in run 889: Run already exists in server. Run id(s): {9006773}
[2018-04-11 22:26:34] Error in run 890: Run already exists in server. Run id(s): {9006770}
[2018-04-11 22:26:34] Started run 897 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:34] Started run 898 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:35] Error in run 891: Run already exists in server. Run id(s): {9006763}
[2018-04-11 22:26:35] Error in run 892: Run already exists in server. Run id(s): {9006774}
[2018-04-11 22:26:35] Started run 899 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:35] Started run 900 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:35] Error in run 893: Run already exists in server. Run id(s): {9006766}
[2018-04-11 22:26:35] Error in run 894: Run already exists in server. Run id(s): {9006777}
[2018-04-11 22:26:35] Started run 901 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:35] Started run 902 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:36] Error in run 895: Run already exists in server. Run id(s): {9006775}
[2018-04-11 22:26:36] Started run 903 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:37] Error in run 896: Run already exists in server. Run id(s): {9006798}
[2018-04-11 22:26:37] Started run 904 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:39] Error in run 897: Run already exists in server. Run id(s): {9006792}
[2018-04-11 22:26:39] Error in run 898: Run already exists in server. Run id(s): {9006802}
[2018-04-11 22:26:39] Started run 905 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:39] Started run 906 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:39] Error in run 899: Run already exists in server. Run id(s): {9006783}
[2018-04-11 22:26:39] Started run 907 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:39] Error in run 900: Run already exists in server. Run id(s): {9006812}
[2018-04-11 22:26:39] Started run 908 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:40] Error in run 901: Run already exists in server. Run id(s): {9006789}
[2018-04-11 22:26:40] Started run 909 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:40] Error in run 902: Run already exists in server. Run id(s): {9006788}
[2018-04-11 22:26:40] Started run 910 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:40] Error in run 903: Run already exists in server. Run id(s): {9006790}
[2018-04-11 22:26:40] Started run 911 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:42] Error in run 904: Run already exists in server. Run id(s): {9006821}
[2018-04-11 22:26:42] Started run 912 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:43] Error in run 905: Run already exists in server. Run id(s): {9006805}
[2018-04-11 22:26:43] Error in run 906: Run already exists in server. Run id(s): {9006816}
[2018-04-11 22:26:43] Started run 913 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:43] Started run 914 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:44] Error in run 907: Run already exists in server. Run id(s): {9006807}
[2018-04-11 22:26:44] Error in run 908: Run already exists in server. Run id(s): {9006808}
[2018-04-11 22:26:44] Started run 915 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:44] Started run 916 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:45] Error in run 909: Run already exists in server. Run id(s): {9006818}
[2018-04-11 22:26:45] Error in run 910: Run already exists in server. Run id(s): {9006826}
[2018-04-11 22:26:45] Started run 917 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:45] Started run 918 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:45] Error in run 911: Run already exists in server. Run id(s): {9006823}
[2018-04-11 22:26:45] Started run 919 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:46] Error in run 912: Run already exists in server. Run id(s): {9006824}
[2018-04-11 22:26:46] Started run 920 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:48] Error in run 913: Run already exists in server. Run id(s): {9006814}
[2018-04-11 22:26:48] Error in run 914: Run already exists in server. Run id(s): {9006848}
[2018-04-11 22:26:48] Started run 921 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:48] Started run 922 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:49] Error in run 915: Run already exists in server. Run id(s): {9006854}
[2018-04-11 22:26:49] Error in run 916: Run already exists in server. Run id(s): {9006834}
[2018-04-11 22:26:49] Started run 923 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:49] Started run 924 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:49] Error in run 917: Run already exists in server. Run id(s): {9006856}
[2018-04-11 22:26:49] Started run 925 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:50] Error in run 918: Run already exists in server. Run id(s): {9006847}
[2018-04-11 22:26:50] Started run 926 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:50] Error in run 919: Run already exists in server. Run id(s): {9006839}
[2018-04-11 22:26:50] Started run 927 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:51] Error in run 920: Run already exists in server. Run id(s): {9006846}
[2018-04-11 22:26:51] Started run 928 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:53] Error in run 921: Run already exists in server. Run id(s): {9006849}
[2018-04-11 22:26:53] Error in run 922: Run already exists in server. Run id(s): {9006867}
[2018-04-11 22:26:53] Started run 929 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:53] Started run 930 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:53] Error in run 923: Run already exists in server. Run id(s): {9006853}
[2018-04-11 22:26:53] Error in run 924: Run already exists in server. Run id(s): {9006892}
[2018-04-11 22:26:53] Started run 931 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:53] Started run 932 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:54] Error in run 925: Run already exists in server. Run id(s): {9006860}
[2018-04-11 22:26:54] Error in run 926: Run already exists in server. Run id(s): {9006857}
[2018-04-11 22:26:54] Started run 933 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:54] Started run 934 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:55] Error in run 927: Run already exists in server. Run id(s): {9006858}
[2018-04-11 22:26:55] Started run 935 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:56] Error in run 928: Run already exists in server. Run id(s): {9006880}
[2018-04-11 22:26:56] Started run 936 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:57] Error in run 929: Run already exists in server. Run id(s): {9006863}
[2018-04-11 22:26:57] Error in run 930: Run already exists in server. Run id(s): {9006878}
[2018-04-11 22:26:57] Started run 937 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:57] Started run 938 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:58] Error in run 931: Run already exists in server. Run id(s): {9006884}
[2018-04-11 22:26:58] Started run 939 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:58] Error in run 932: Run already exists in server. Run id(s): {9006872}
[2018-04-11 22:26:58] Started run 940 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:59] Error in run 933: Run already exists in server. Run id(s): {9006888}
[2018-04-11 22:26:59] Error in run 934: Run already exists in server. Run id(s): {9006901}
[2018-04-11 22:26:59] Started run 941 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:59] Started run 942 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:26:59] Error in run 935: Run already exists in server. Run id(s): {9006886}
[2018-04-11 22:26:59] Started run 943 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:00] Error in run 936: Run already exists in server. Run id(s): {9006895}
[2018-04-11 22:27:00] Started run 944 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:02] Error in run 937: Run already exists in server. Run id(s): {9006915}
[2018-04-11 22:27:02] Error in run 938: Run already exists in server. Run id(s): {9006899}
[2018-04-11 22:27:02] Started run 945 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:02] Started run 946 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:03] Error in run 940: Run already exists in server. Run id(s): {9006919}
[2018-04-11 22:27:03] Error in run 939: Run already exists in server. Run id(s): {9006922}
[2018-04-11 22:27:03] Started run 947 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:03] Started run 948 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:04] Error in run 942: Run already exists in server. Run id(s): {9006926}
[2018-04-11 22:27:04] Error in run 941: Run already exists in server. Run id(s): {9006909}
[2018-04-11 22:27:04] Started run 949 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:04] Error in run 943: Run already exists in server. Run id(s): {9006908}
[2018-04-11 22:27:04] Started run 950 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:04] Started run 951 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:05] Error in run 944: Run already exists in server. Run id(s): {9006940}
[2018-04-11 22:27:05] Started run 952 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:07] Error in run 945: Run already exists in server. Run id(s): {9006924}
[2018-04-11 22:27:07] Error in run 946: Run already exists in server. Run id(s): {9006933}
[2018-04-11 22:27:07] Started run 953 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:07] Started run 954 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:07] Error in run 947: Run already exists in server. Run id(s): {9006930}
[2018-04-11 22:27:07] Started run 955 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:08] Error in run 948: Run already exists in server. Run id(s): {9006936}
[2018-04-11 22:27:08] Started run 956 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:08] Error in run 949: Run already exists in server. Run id(s): {9006937}
[2018-04-11 22:27:08] Started run 957 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:08] Error in run 950: Run already exists in server. Run id(s): {9006949}
[2018-04-11 22:27:08] Error in run 951: Run already exists in server. Run id(s): {9006956}
[2018-04-11 22:27:08] Started run 958 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:08] Started run 959 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:10] Error in run 952: Run already exists in server. Run id(s): {9006942}
[2018-04-11 22:27:10] Started run 960 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:11] Error in run 953: Run already exists in server. Run id(s): {9006973}
[2018-04-11 22:27:11] Error in run 954: Run already exists in server. Run id(s): {9006969}
[2018-04-11 22:27:11] Started run 961 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:11] Started run 962 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:12] Error in run 955: Run already exists in server. Run id(s): {9006955}
[2018-04-11 22:27:12] Error in run 956: Run already exists in server. Run id(s): {9006951}
[2018-04-11 22:27:12] Started run 963 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:12] Started run 964 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:13] Error in run 957: Run already exists in server. Run id(s): {9006960}
[2018-04-11 22:27:13] Started run 965 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:13] Error in run 958: Run already exists in server. Run id(s): {9006958}
[2018-04-11 22:27:13] Error in run 959: Run already exists in server. Run id(s): {9006976}
[2018-04-11 22:27:13] Started run 966 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:13] Started run 967 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:14] Error in run 960: Run already exists in server. Run id(s): {9006968}
[2018-04-11 22:27:14] Started run 968 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:16] Error in run 961: Run already exists in server. Run id(s): {9006980}
[2018-04-11 22:27:16] Error in run 962: Run already exists in server. Run id(s): {9006978}
[2018-04-11 22:27:16] Started run 969 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:16] Started run 970 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:17] Error in run 963: Run already exists in server. Run id(s): {9006975}
[2018-04-11 22:27:17] Error in run 964: Run already exists in server. Run id(s): {9006999}
[2018-04-11 22:27:17] Started run 971 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:17] Started run 972 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:18] Error in run 965: Run already exists in server. Run id(s): {9006982}
[2018-04-11 22:27:18] Started run 973 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:18] Error in run 966: Run already exists in server. Run id(s): {9006997}
[2018-04-11 22:27:18] Started run 974 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:18] Error in run 967: Run already exists in server. Run id(s): {9007000}
[2018-04-11 22:27:18] Started run 975 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:19] Error in run 968: Run already exists in server. Run id(s): {9007001}
[2018-04-11 22:27:19] Started run 976 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:21] Error in run 969: Run already exists in server. Run id(s): {9007002}
[2018-04-11 22:27:21] Started run 977 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:21] Error in run 970: Run already exists in server. Run id(s): {9006988}
[2018-04-11 22:27:21] Started run 978 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:22] Error in run 971: Run already exists in server. Run id(s): {9007009}
[2018-04-11 22:27:22] Error in run 972: Run already exists in server. Run id(s): {9006996}
[2018-04-11 22:27:22] Started run 979 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:22] Started run 980 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:22] Error in run 973: Run already exists in server. Run id(s): {9007005}
[2018-04-11 22:27:22] Error in run 974: Run already exists in server. Run id(s): {9007016}
[2018-04-11 22:27:22] Started run 981 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:23] Error in run 975: Run already exists in server. Run id(s): {9007034}
[2018-04-11 22:27:22] Started run 982 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:23] Started run 983 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:24] Error in run 976: Run already exists in server. Run id(s): {9007028}
[2018-04-11 22:27:24] Started run 984 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:25] Error in run 977: Run already exists in server. Run id(s): {9007014}
[2018-04-11 22:27:25] Error in run 978: Run already exists in server. Run id(s): {9007038}
[2018-04-11 22:27:25] Started run 985 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:25] Started run 986 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:26] Error in run 979: Run already exists in server. Run id(s): {9007019}
[2018-04-11 22:27:26] Started run 987 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:26] Error in run 980: Run already exists in server. Run id(s): {9007022}
[2018-04-11 22:27:26] Started run 988 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:27] Error in run 981: Run already exists in server. Run id(s): {9007025}
[2018-04-11 22:27:27] Error in run 982: Run already exists in server. Run id(s): {9007057}
[2018-04-11 22:27:27] Started run 989 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:27] Error in run 983: Run already exists in server. Run id(s): {9007032}
[2018-04-11 22:27:27] Started run 990 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:27] Started run 991 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:29] Error in run 984: Run already exists in server. Run id(s): {9007047}
[2018-04-11 22:27:29] Started run 992 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:30] Error in run 985: Run already exists in server. Run id(s): {9007030}
[2018-04-11 22:27:30] Error in run 986: Run already exists in server. Run id(s): {9007051}
[2018-04-11 22:27:30] Started run 993 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:30] Started run 994 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:31] Error in run 987: Run already exists in server. Run id(s): {9007055}
[2018-04-11 22:27:31] Error in run 988: Run already exists in server. Run id(s): {9007044}
[2018-04-11 22:27:31] Started run 995 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:31] Started run 996 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:32] Error in run 989: Run already exists in server. Run id(s): {9007050}
[2018-04-11 22:27:32] Error in run 990: Run already exists in server. Run id(s): {9007056}
[2018-04-11 22:27:32] Started run 997 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:32] Error in run 991: Run already exists in server. Run id(s): {9007060}
[2018-04-11 22:27:32] Started run 998 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:32] Started run 999 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:33] Error in run 992: Run already exists in server. Run id(s): {9007067}
[2018-04-11 22:27:33] Started run 1000 on task 9981, dataset 'cnae-9'.
[2018-04-11 22:27:35] Error in run 993: Run already exists in server. Run id(s): {9007077}
[2018-04-11 22:27:35] Error in run 994: Run already exists in server. Run id(s): {9007080}
[2018-04-11 22:27:36] Error in run 995: Run already exists in server. Run id(s): {9007084}
[2018-04-11 22:27:36] Error in run 996: Run already exists in server. Run id(s): {9007074}
[2018-04-11 22:27:36] Error in run 997: Run already exists in server. Run id(s): {9007072}
[2018-04-11 22:27:36] Error in run 998: Run already exists in server. Run id(s): {9007069}
[2018-04-11 22:27:37] Error in run 999: Run already exists in server. Run id(s): {9007081}
[2018-04-11 22:27:38] Error in run 1000: Run already exists in server. Run id(s): {9007107}
